<!DOCTYPE html>
<html lang="en"><head><script src="https://pagead2.googlesyndication.com/pagead/js/rum.js"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><title>
	Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid 
Gene Sequences and Transcript Processing Pathways in the Fucoxanthin 
Dinoflagellate Karlodinium veneficum | Molecular Biology and Evolution |
 Oxford Academic
</title>   
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <!-- Meta -->
    <meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="https://academic-oup-com.insb.bib.cnrs.fr/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic-oup-com.insb.bib.cnrs.fr/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic-oup-com.insb.bib.cnrs.fr/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic-oup-com.insb.bib.cnrs.fr/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic-oup-com.insb.bib.cnrs.fr/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic-oup-com.insb.bib.cnrs.fr/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">

    <!-- Stylesheets -->
    
<link href="css.css" type="text/css" rel="stylesheet">
<link href="globalCSS.css" type="text/css" rel="stylesheet">
<link href="cookiePolicyCSS.css" type="text/css" rel="stylesheet">
<link href="chartist.css" type="text/css" rel="stylesheet">

        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-304756074.css" rel="stylesheet" type="text/css">
    

    <!-- Scripts -->
    
<script async="" src="analytics.js"></script><script src="jquery-1.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'mbe';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic-oup-com.insb.bib.cnrs.fr/mbe/article/31/9/2376/2925780","@type":"ScholarlyArticle","name":"Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid Gene Sequences and Transcript Processing Pathways in the Fucoxanthin Dinoflagellate  Karlodinium veneficum ","datePublished":"2014-06-12","isPartOf":{"@id":"https://academic-oup-com.insb.bib.cnrs.fr/mbe/issue/31/9","@type":"PublicationIssue","issueNumber":"9","datePublished":"2014-09-01","isPartOf":{"@id":"https://academic-oup-com.insb.bib.cnrs.fr/mbe","@type":"Periodical","name":"Molecular Biology and Evolution","issn":["1537-1719"]}},"url":"http://dx.doi.org.insb.bib.cnrs.fr/10.1093/molbev/msu189","keywords":["dinoflagellate","chloroplast","RNA processing","editing","polyuridylylation","minicircle"],"inLanguage":"en","copyrightHolder":"Society for Molecular Biology and Evolution","copyrightYear":"2017","publisher":"Oxford University Press","sameAs":"https://academic-oup-com.insb.bib.cnrs.fr/mbe/article/31/9/2376/2925780","author":[{"name":"Richardson, Elisabeth","affiliation":"Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom","@type":"Person"},{"name":"Dorrell, Richard G.","affiliation":"Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom","@type":"Person"},{"name":"Howe, Christopher J.","affiliation":"Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom","@type":"Person"}],"description":"Plastids utilize a complex gene expression machinery, which has coevolved with the underlying genome sequence. Relatively, little is known about the genome-wide evolution of transcript processing in algal plastids that have undergone complex endosymbiotic events. We present the first genome-wide study of transcript processing in a plastid acquired through serial endosymbiosis, in the fucoxanthin-containing dinoflagellate Karlodinium veneficum. The fucoxanthin dinoflagellate plastid has an extremely divergent genome and utilizes two unusual transcript processing pathways, 3′-poly(U) tail addition and sequence editing, which were acquired following the serial endosymbiosis event. We demonstrate that poly(U) addition and sequence editing are widespread features across the Karl. veneficum plastid transcriptome, whereas other dinoflagellate plastid lineages that have arisen through independent serial endosymbiosis events do not utilize either RNA processing pathway. These pathways constrain the effects of divergent sequence evolution in fucoxanthin plastids, for example by correcting mutations in the genomic sequence that would otherwise be deleterious, and are specifically associated with transcripts that encode functional plastid proteins over transcripts of recently generated pseudogenes. These pathways may have additionally facilitated divergent evolution within the Karl. veneficum plastid. Transcript editing, for example, has contributed to the evolution of a novel C-terminal sequence extension on the Karl. veneficum AtpA protein. We furthermore provide the first complete sequence of an episomal minicircle in a fucoxanthin dinoflagellate plastid, which contains the dnaK gene, and gives rise to polyuridylylated and edited transcripts. Our results indicate that RNA processing in fucoxanthin dinoflagellate plastids is evolutionarily dynamic, coevolving with the underlying genome sequence.","pageStart":"2376","pageEnd":"2386","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/31/9/0/m_mbe31_9.cover.gif?Expires=1575979984&Signature=Q12oZPyOlnxdln8-80ssmPttTlXV8lvhcF17TVXcCMQHY9jmmJccNz7dUM1JwfoL7GAIjPi4WyftdpGucIGg67ePLQrYnwaSQKYGwUmBRa2QpbNdnOyQ0JBqUxlfoU3CaeVbrG-iJxaBE6dr9j8C~q5dNZvyPFbFMSiEv~CWSTTEY~BXzjpL7gD3FcEA5RJ15m5uH~T4ptKwOwalQmVs3TVNE40HOaY9Mi5Oj9VCOsurCwTcpAkzX3MRHz8nNcRZU7r6PE0WFgM7fRMP~fkN81nzMuqFpLg89aDH0evEAv1dQlGdRODcfUl1CoIl2AXtTKbxmPMbw8f-91V6N5X2Pg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q","headline":"Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid Gene Sequences and Transcript Processing Pathways in the Fucoxanthin Dinoflagellate  Karlodinium veneficum ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/31/9/0/m_mbe31_9.cover.gif?Expires=1575979984&Signature=Q12oZPyOlnxdln8-80ssmPttTlXV8lvhcF17TVXcCMQHY9jmmJccNz7dUM1JwfoL7GAIjPi4WyftdpGucIGg67ePLQrYnwaSQKYGwUmBRa2QpbNdnOyQ0JBqUxlfoU3CaeVbrG-iJxaBE6dr9j8C~q5dNZvyPFbFMSiEv~CWSTTEY~BXzjpL7gD3FcEA5RJ15m5uH~T4ptKwOwalQmVs3TVNE40HOaY9Mi5Oj9VCOsurCwTcpAkzX3MRHz8nNcRZU7r6PE0WFgM7fRMP~fkN81nzMuqFpLg89aDH0evEAv1dQlGdRODcfUl1CoIl2AXtTKbxmPMbw8f-91V6N5X2Pg__&Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid Gene Sequences and Transcript Processing Pathways in the Fucoxanthin Dinoflagellate  Karlodinium veneficum  | Molecular Biology and Evolution | Oxford Academic">
    <meta property="og:description" content="Plastids utilize a complex gene expression machinery, which has coevolved with the underlying genome sequence. Relatively, little is known about the genome-wide evolution of transcript processing in algal plastids that have undergone complex endosymbiotic events. We present the first genome-wide study of transcript processing in a plastid acquired through serial endosymbiosis, in the fucoxanthin-containing dinoflagellate Karlodinium veneficum. The fucoxanthin dinoflagellate plastid has an extremely divergent genome and utilizes two unusual transcript processing pathways, 3′-poly(U) tail addition and sequence editing, which were acquired following the serial endosymbiosis event. We demonstrate that poly(U) addition and sequence editing are widespread features across the Karl. veneficum plastid transcriptome, whereas other dinoflagellate plastid lineages that have arisen through independent serial endosymbiosis events do not utilize either RNA processing pathway. These pathways constrain the effects of divergent sequence evolution in fucoxanthin plastids, for example by correcting mutations in the genomic sequence that would otherwise be deleterious, and are specifically associated with transcripts that encode functional plastid proteins over transcripts of recently generated pseudogenes. These pathways may have additionally facilitated divergent evolution within the Karl. veneficum plastid. Transcript editing, for example, has contributed to the evolution of a novel C-terminal sequence extension on the Karl. veneficum AtpA protein. We furthermore provide the first complete sequence of an episomal minicircle in a fucoxanthin dinoflagellate plastid, which contains the dnaK gene, and gives rise to polyuridylylated and edited transcripts. Our results indicate that RNA processing in fucoxanthin dinoflagellate plastids is evolutionarily dynamic, coevolving with the underlying genome sequence.">
    <meta property="og:url" content="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article/31/9/2376/2925780">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/34/12/2/m_molbiolevol_34_12cover.png?Expires=1575979984&amp;Signature=JR2w3ybdiL1sOtA3Wra03JPEEgKDMdKjnW7YP9Uo5clbc6jMiAjrVaX70rX9MqJQlH7IVuQhFUQvTF0EICX30qnlS~2xEpkq5BuRu4I4bPciFiF-hjwyBusxUK23WlBpxuAkaRoJ1EwC2D3Eojy7j7e~bqJVWsSOSRGyO9~sQjBy38u5mZcre-RsmhWHDCM9nD3ynK8y7AqSUWqwpRQeF0-oo8KqU58ADFgM4kUbIvLlPzxEm2KzvMkvEMKD8KdkZbIncWy~ScXZhocQGABQeopg7zueRyfbdfmB0UM8T~4XdNKq-K4~RfGSZBXxuBYgVEJDtCrlE-sRpwIs7pbWDA__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">
    <meta property="og:updated_time" content="2014-06-12">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Richardson, Elisabeth"><meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"><meta name="citation_author" content="Dorrell, Richard G."><meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"><meta name="citation_author" content="Howe, Christopher J."><meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom"><meta name="citation_title" content="Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid Gene Sequences and Transcript Processing Pathways in the Fucoxanthin Dinoflagellate Karlodinium veneficum"><meta name="citation_firstpage" content="2376"><meta name="citation_lastpage" content="2386"><meta name="citation_doi" content="10.1093/molbev/msu189"><meta name="citation_journal_title" content="Molecular Biology and Evolution"><meta name="citation_journal_abbrev" content="Mol Biol Evol"><meta name="citation_volume" content="31"><meta name="citation_issue" content="9"><meta name="citation_publication_date" content="2014/09/01"><meta name="citation_issn" content="0737-4038"><meta name="citation_publisher" content="Oxford University Press"><meta name="citation_reference" content="citation_title=Two CRM protein subfamilies cooperate in the splicing of group IIB introns in chloroplasts; RNA;  citation_year=2008;  citation_volume=14;  citation_pages=2319-2332; "><meta name="citation_reference" content="citation_title=Dinoflagellate expressed indicate massive transfer to the nuclear genome sequence tag data of chloroplast genes; Protist;  citation_year=2004;  citation_volume=155;  citation_pages=65-78; "><meta name="citation_reference" content="citation_title=Polyuridylylation and processing of transcripts from multiple gene minicircles in chloroplasts of the dinoflagellate Amphidinium carterae; Plant Mol Biol.;  citation_year=2012;  citation_volume=79;  citation_pages=347-357; "><meta name="citation_reference" content="citation_title=Minicircular plastid DNA in the dinoflagellate Amphidinium operculatum; Mol Gen Genet.;  citation_year=2000;  citation_volume=263;  citation_pages=152-158; "><meta name="citation_reference" content="citation_title=Phylogenomic evidence for separate acquisition of plastids in cryptophytes, haptophytes, and stramenopiles; Mol Biol Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=1698-1709; "><meta name="citation_reference" content="citation_title=On the identity of Karlodinium veneficum and description of Karlodinium armiger sp nov (Dinophyceae), based on light and electron microscopy, nuclear-encoded LSU rDNA, and pigment composition; J Phycol.;  citation_year=2006;  citation_volume=42;  citation_pages=170-193; "><meta name="citation_reference" content="citation_title=Endosymbiotic gene transfer in tertiary plastid-containing dinoflagellates; Eukaryot Cell.;  citation_year=2014;  citation_volume=13;  citation_pages=246-255; "><meta name="citation_reference" content="citation_title=Targeted metagenomics and ecology of globally important uncultured eukaryotic phytoplankton; Proc Natl Acad Sci U S A.;  citation_year=2010;  citation_volume=107;  citation_pages=14679-14684; "><meta name="citation_reference" content="citation_title=Substitutional editing of Heterocapsa triquetra chloroplast transcripts and a folding model for its divergent chloroplast 16S rRNA; Gene;  citation_year=2009;  citation_volume=442;  citation_pages=73-80; "><meta name="citation_reference" content="citation_title=Evolution of chloroplast transcript processing in Plasmodium and its chromerid algal relatives; PLoS Genet.;  citation_year=2014;  citation_volume=10;  citation_pages=e1004008"><meta name="citation_reference" content="citation_title=Functional remodeling of RNA processing in replacement chloroplasts by pathways retained from their predecessors; Proc Natl Acad Sci U S A.;  citation_year=2012;  citation_volume=109;  citation_pages=18879-18884; "><meta name="citation_reference" content="citation_title=Genome fragmentation is not confined to the peridinin plastid in dinoflagellates; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e38809"><meta name="citation_reference" content="citation_title=The evolution of RNA editing and pentatricopeptide repeat genes; New Phytol.;  citation_year=2011;  citation_volume=191;  citation_pages=37-47; "><meta name="citation_reference" content="citation_title=Genome evolution of a tertiary dinoflagellate plastid; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e19132"><meta name="citation_reference" content="citation_title=Chloroplast genomes of photosynthetic eukaryotes; Plant J.;  citation_year=2011;  citation_volume=66;  citation_pages=34-44; "><meta name="citation_reference" content="citation_title=Molecular evolution of pentatricopeptide repeat genes reveals truncation in species lacking an editing target and structural domains under distinct selective pressures; BMC Evol Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=13"><meta name="citation_reference" content="citation_title=The remarkable chloroplast genome of dinoflagellates; J Exp Bot.;  citation_year=2008;  citation_volume=59;  citation_pages=1035-1045; "><meta name="citation_reference" content="citation_title=Identification of transcribed and persistent variants of the psbA gene carried by plastid minicircles in a dinoflagellate; Curr Genet.;  citation_year=2009;  citation_volume=55;  citation_pages=583-591; "><meta name="citation_reference" content="citation_title=The complete plastid genomes of the two ‘dinotoms’ Durinskia baltica and Kryptoperidinium foliaceum; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e10711"><meta name="citation_reference" content="citation_title=A tertiary plastid gains RNA editing in its new host; Mol Biol Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=788-792; "><meta name="citation_reference" content="citation_title=A common red algal origin of the apicomplexan, dinoflagellate, and heterokont plastids; Proc Natl Acad Sci U S A.;  citation_year=2010;  citation_volume=107;  citation_pages=10949-10954; "><meta name="citation_reference" content="citation_title=Split photosystem protein, linear-mapping topology, and growth of structural complexity in the plastid genome of Chromera velia; Mol Biol Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2447-2462; "><meta name="citation_reference" content="citation_title=Polyadenylation-assisted RNA degradation processes in plants; Trends Plant Sci.;  citation_year=2009;  citation_volume=14;  citation_pages=497-504; "><meta name="citation_reference" content="citation_title=Specificity for activase is changed by a Pro-89 to Arg substitution in the large subunit of ribulose-1,5-bisphosphate carboxylase/oxygenase; J Biol Chem.;  citation_year=1997;  citation_volume=272;  citation_pages=17033-17037; "><meta name="citation_reference" content="citation_title=Two residues of rubisco activase involved in recognition of the rubisco substrate; J Biol Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=24864-24869; "><meta name="citation_reference" content="citation_title=Efficient isolation and mapping of Arabidopsis thaliana T-DNA insert junctions by thermal asymmetric interlaced PCR; Plant J.;  citation_year=1995;  citation_volume=8;  citation_pages=457-463; "><meta name="citation_reference" content="citation_title=Green-colored plastids in the dinoflagellate genus Lepidodinium are of core chlorophyte origin; Protist;  citation_year=2011;  citation_volume=162;  citation_pages=268-276; "><meta name="citation_reference" content="citation_title=A phylogenetic mosaic plastid proteome and unusual plastid-targeting signals in the green-colored dinoflagellate Lepidodinium chlorophorum; BMC Evol Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=191"><meta name="citation_reference" content="citation_title=Massive gene transfer and extensive RNA editing of a symbiotic dinoflagellate plastid genome; Genom Biol Evol.;  citation_year=2014;  citation_volume=6;  citation_pages=1408-1422; "><meta name="citation_reference" content="citation_title=Organization of the mitochondrial genome in the dinoflagellate Amphidinium carterae; Mol Biol Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1528-1536; "><meta name="citation_reference" content="citation_title=Identification and transcription of transfer RNA genes in dinoflagellate plastid minicircles; Gene;  citation_year=2007;  citation_volume=392;  citation_pages=291-298; "><meta name="citation_reference" content="citation_title=The complete plastid genome sequence of the haptophyte Emiliania huxleyi: a comparison to other plastid genomes; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=151-156; "><meta name="citation_reference" content="citation_title=Heterotachy processes in rhodophyte-derived secondhand plastid genes: implications for addressing the origin and evolution of dinoflagellate plastids; Mol Biol Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1504-1515; "><meta name="citation_reference" content="citation_title=Serial replacement of diatom endosymbionts in two freshwater dinoflagenates, Peridiniopsis spp. (Peridiniales, Dinophyceae); Phycologia;  citation_year=2008;  citation_volume=47;  citation_pages=41-53; "><meta name="citation_reference" content="citation_title=Origins of plastids and glyceraldehyde-3-phosphate dehydrogenase genes in the green-colored dinoflagellate Lepidodinium chlorophorum; Gene;  citation_year=2008;  citation_volume=410;  citation_pages=26-36; "><meta name="citation_reference" content="citation_title=Preliminary phylogenetic analysis of plastid-encoded genes from an anomalously pigmented dinoflagellate Gymnodinium mikimotoi (Gymnodiniales, Dinophyta); Phycol Res.;  citation_year=1999;  citation_volume=47;  citation_pages=257-262; "><meta name="citation_reference" content="citation_title=WAR: Webserver for aligning structural RNAs; Nucl Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=79-84; "><meta name="citation_reference" content="citation_title=Rampant polyuridylylation of plastid gene transcripts in the dinoflagellate Lingulodinium; Nucl Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=613-619; "><meta name="citation_reference" content="citation_title=Substitutional editing of transcripts from genes of cyanobacterial origin in the dinoflagellate Ceratium horridum; FEBS Lett.;  citation_year=2004;  citation_volume=577;  citation_pages=535-538; "><meta name="citation_reference" content="citation_title=Single gene circles in dinoflagellate chloroplast genomes; Nature;  citation_year=1999;  citation_volume=400;  citation_pages=155-159; "><meta name="citation_reference" content="citation_title=Mfold web server for nucleic acid folding and hybridization prediction; Nucl Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3406-3415; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article-pdf/31/9/2376/13171499/msu189.pdf"><meta name="description" content="Abstract.  Plastids utilize a complex gene expression machinery, which has coevolved with the underlying genome sequence. Relatively, little is known about the ">
        <meta name="citation_xml_url" content="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article-xml/31/9/2376/2925780">



    <link rel="canonical" href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article/31/9/2376/2925780">


        <!-- Start: GPT Async -->

<script async="async" src="gpt.js"></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type="text/javascript">
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    

    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    googletag.enableServices();
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global-oup-com.insb.bib.cnrs.fr/cookiepolicy/" target="_blank">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=molbev&authentication_method=IP&authzrequired=false&account_id=2790002&account_list=2790002,4443063&authnips=80.12.27.138&doi=10.1093/molbev/msu189';
    </script>
<script id="altmetric-embed-js" src="altmetric_badges-eb187a407f01c9f3bbcda1dddf27884f.js"></script><style></style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><link rel="preload" href="integrator_002.js"><script type="text/javascript" src="integrator_002.js"></script><script src="pubads_impl_170.js" async=""></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_stumbleupon_badge,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-14/html/container.html"><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-2c47105b6381604898bbf8ae8a680350.css"></head>

<body data-sitename="molecularbiologyandevolution" class="off-canvas pg_article" theme-mbe="" style="cursor: default;"><div id="MathJax_Message" style="display: none;"></div>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 10});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2925780" id="webform" novalidate="novalidate">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwULLTE5MDQ5NDk5MjIPZBYCZg9kFgJmD2QWBGYPZBYCAg8PZBYCAgEPZBYCZg8WAh4EVGV4dAWlWzxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmljaGFyZHNvbiwgRWxpc2FiZXRoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9jaGVtaXN0cnksIFVuaXZlcnNpdHkgb2YgQ2FtYnJpZGdlLCBDYW1icmlkZ2UsIFVuaXRlZCBLaW5nZG9tIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRG9ycmVsbCwgUmljaGFyZCBHLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvY2hlbWlzdHJ5LCBVbml2ZXJzaXR5IG9mIENhbWJyaWRnZSwgQ2FtYnJpZGdlLCBVbml0ZWQgS2luZ2RvbSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhvd2UsIENocmlzdG9waGVyIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9jaGVtaXN0cnksIFVuaXZlcnNpdHkgb2YgQ2FtYnJpZGdlLCBDYW1icmlkZ2UsIFVuaXRlZCBLaW5nZG9tIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtV2lkZSBUcmFuc2NyaXB0IFByb2ZpbGluZyBSZXZlYWxzIHRoZSBDb2V2b2x1dGlvbiBvZiBQbGFzdGlkIEdlbmUgU2VxdWVuY2VzIGFuZCBUcmFuc2NyaXB0IFByb2Nlc3NpbmcgUGF0aHdheXMgaW4gdGhlIEZ1Y294YW50aGluIERpbm9mbGFnZWxsYXRlIEthcmxvZGluaXVtIHZlbmVmaWN1bSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIzNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIzODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL21vbGJldi9tc3UxODkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEV2b2x1dGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iTW9sIEJpb2wgRXZvbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMDkvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMDczNy00MDM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gQ1JNIHByb3RlaW4gc3ViZmFtaWxpZXMgY29vcGVyYXRlIGluIHRoZSBzcGxpY2luZyBvZiBncm91cCBJSUIgaW50cm9ucyBpbiBjaGxvcm9wbGFzdHM7IFJOQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIzMTktMjMzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlub2ZsYWdlbGxhdGUgZXhwcmVzc2VkIGluZGljYXRlIG1hc3NpdmUgdHJhbnNmZXIgdG8gdGhlIG51Y2xlYXIgZ2Vub21lIHNlcXVlbmNlIHRhZyBkYXRhIG9mIGNobG9yb3BsYXN0IGdlbmVzOyBQcm90aXN0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTY1LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5dXJpZHlseWxhdGlvbiBhbmQgcHJvY2Vzc2luZyBvZiB0cmFuc2NyaXB0cyBmcm9tIG11bHRpcGxlIGdlbmUgbWluaWNpcmNsZXMgaW4gY2hsb3JvcGxhc3RzIG9mIHRoZSBkaW5vZmxhZ2VsbGF0ZSBBbXBoaWRpbml1bSBjYXJ0ZXJhZTsgUGxhbnQgTW9sIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9MzQ3LTM1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWluaWNpcmN1bGFyIHBsYXN0aWQgRE5BIGluIHRoZSBkaW5vZmxhZ2VsbGF0ZSBBbXBoaWRpbml1bSBvcGVyY3VsYXR1bTsgTW9sIEdlbiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjYzOyAgY2l0YXRpb25fcGFnZXM9MTUyLTE1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5vbWljIGV2aWRlbmNlIGZvciBzZXBhcmF0ZSBhY3F1aXNpdGlvbiBvZiBwbGFzdGlkcyBpbiBjcnlwdG9waHl0ZXMsIGhhcHRvcGh5dGVzLCBhbmQgc3RyYW1lbm9waWxlczsgTW9sIEJpb2wgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xNjk4LTE3MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHRoZSBpZGVudGl0eSBvZiBLYXJsb2Rpbml1bSB2ZW5lZmljdW0gYW5kIGRlc2NyaXB0aW9uIG9mIEthcmxvZGluaXVtIGFybWlnZXIgc3Agbm92IChEaW5vcGh5Y2VhZSksIGJhc2VkIG9uIGxpZ2h0IGFuZCBlbGVjdHJvbiBtaWNyb3Njb3B5LCBudWNsZWFyLWVuY29kZWQgTFNVIHJETkEsIGFuZCBwaWdtZW50IGNvbXBvc2l0aW9uOyBKIFBoeWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xNzAtMTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvc3ltYmlvdGljIGdlbmUgdHJhbnNmZXIgaW4gdGVydGlhcnkgcGxhc3RpZC1jb250YWluaW5nIGRpbm9mbGFnZWxsYXRlczsgRXVrYXJ5b3QgQ2VsbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yNDYtMjU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBtZXRhZ2Vub21pY3MgYW5kIGVjb2xvZ3kgb2YgZ2xvYmFsbHkgaW1wb3J0YW50IHVuY3VsdHVyZWQgZXVrYXJ5b3RpYyBwaHl0b3BsYW5rdG9uOyBQcm9jIE5hdGwgQWNhZCBTY2kgVSBTIEEuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTE0Njc5LTE0Njg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWJzdGl0dXRpb25hbCBlZGl0aW5nIG9mIEhldGVyb2NhcHNhIHRyaXF1ZXRyYSBjaGxvcm9wbGFzdCB0cmFuc2NyaXB0cyBhbmQgYSBmb2xkaW5nIG1vZGVsIGZvciBpdHMgZGl2ZXJnZW50IGNobG9yb3BsYXN0IDE2UyByUk5BOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ0MjsgIGNpdGF0aW9uX3BhZ2VzPTczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgY2hsb3JvcGxhc3QgdHJhbnNjcmlwdCBwcm9jZXNzaW5nIGluIFBsYXNtb2RpdW0gYW5kIGl0cyBjaHJvbWVyaWQgYWxnYWwgcmVsYXRpdmVzOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA0MDA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCByZW1vZGVsaW5nIG9mIFJOQSBwcm9jZXNzaW5nIGluIHJlcGxhY2VtZW50IGNobG9yb3BsYXN0cyBieSBwYXRod2F5cyByZXRhaW5lZCBmcm9tIHRoZWlyIHByZWRlY2Vzc29yczsgUHJvYyBOYXRsIEFjYWQgU2NpIFUgUyBBLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xODg3OS0xODg4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGZyYWdtZW50YXRpb24gaXMgbm90IGNvbmZpbmVkIHRvIHRoZSBwZXJpZGluaW4gcGxhc3RpZCBpbiBkaW5vZmxhZ2VsbGF0ZXM7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzg4MDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIFJOQSBlZGl0aW5nIGFuZCBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgZ2VuZXM7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE5MTsgIGNpdGF0aW9uX3BhZ2VzPTM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZXZvbHV0aW9uIG9mIGEgdGVydGlhcnkgZGlub2ZsYWdlbGxhdGUgcGxhc3RpZDsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxOTEzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNobG9yb3BsYXN0IGdlbm9tZXMgb2YgcGhvdG9zeW50aGV0aWMgZXVrYXJ5b3RlczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjY7ICBjaXRhdGlvbl9wYWdlcz0zNC00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGV2b2x1dGlvbiBvZiBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgZ2VuZXMgcmV2ZWFscyB0cnVuY2F0aW9uIGluIHNwZWNpZXMgbGFja2luZyBhbiBlZGl0aW5nIHRhcmdldCBhbmQgc3RydWN0dXJhbCBkb21haW5zIHVuZGVyIGRpc3RpbmN0IHNlbGVjdGl2ZSBwcmVzc3VyZXM7IEJNQyBFdm9sIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVtYXJrYWJsZSBjaGxvcm9wbGFzdCBnZW5vbWUgb2YgZGlub2ZsYWdlbGxhdGVzOyBKIEV4cCBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9MTAzNS0xMDQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiB0cmFuc2NyaWJlZCBhbmQgcGVyc2lzdGVudCB2YXJpYW50cyBvZiB0aGUgcHNiQSBnZW5lIGNhcnJpZWQgYnkgcGxhc3RpZCBtaW5pY2lyY2xlcyBpbiBhIGRpbm9mbGFnZWxsYXRlOyBDdXJyIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTU4My01OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBwbGFzdGlkIGdlbm9tZXMgb2YgdGhlIHR3byDigJhkaW5vdG9tc+KAmSBEdXJpbnNraWEgYmFsdGljYSBhbmQgS3J5cHRvcGVyaWRpbml1bSBmb2xpYWNldW07IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTA3MTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRlcnRpYXJ5IHBsYXN0aWQgZ2FpbnMgUk5BIGVkaXRpbmcgaW4gaXRzIG5ldyBob3N0OyBNb2wgQmlvbCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTc4OC03OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tbW9uIHJlZCBhbGdhbCBvcmlnaW4gb2YgdGhlIGFwaWNvbXBsZXhhbiwgZGlub2ZsYWdlbGxhdGUsIGFuZCBoZXRlcm9rb250IHBsYXN0aWRzOyBQcm9jIE5hdGwgQWNhZCBTY2kgVSBTIEEuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTEwOTQ5LTEwOTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpdCBwaG90b3N5c3RlbSBwcm90ZWluLCBsaW5lYXItbWFwcGluZyB0b3BvbG9neSwgYW5kIGdyb3d0aCBvZiBzdHJ1Y3R1cmFsIGNvbXBsZXhpdHkgaW4gdGhlIHBsYXN0aWQgZ2Vub21lIG9mIENocm9tZXJhIHZlbGlhOyBNb2wgQmlvbCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI0NDctMjQ2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seWFkZW55bGF0aW9uLWFzc2lzdGVkIFJOQSBkZWdyYWRhdGlvbiBwcm9jZXNzZXMgaW4gcGxhbnRzOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQ5Ny01MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZmljaXR5IGZvciBhY3RpdmFzZSBpcyBjaGFuZ2VkIGJ5IGEgUHJvLTg5IHRvIEFyZyBzdWJzdGl0dXRpb24gaW4gdGhlIGxhcmdlIHN1YnVuaXQgb2YgcmlidWxvc2UtMSw1LWJpc3Bob3NwaGF0ZSBjYXJib3h5bGFzZS9veHlnZW5hc2U7IEogQmlvbCBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0xNzAzMy0xNzAzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIHJlc2lkdWVzIG9mIHJ1YmlzY28gYWN0aXZhc2UgaW52b2x2ZWQgaW4gcmVjb2duaXRpb24gb2YgdGhlIHJ1YmlzY28gc3Vic3RyYXRlOyBKIEJpb2wgQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjgwOyAgY2l0YXRpb25fcGFnZXM9MjQ4NjQtMjQ4Njk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBpc29sYXRpb24gYW5kIG1hcHBpbmcgb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmEgVC1ETkEgaW5zZXJ0IGp1bmN0aW9ucyBieSB0aGVybWFsIGFzeW1tZXRyaWMgaW50ZXJsYWNlZCBQQ1I7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00NTctNDYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HcmVlbi1jb2xvcmVkIHBsYXN0aWRzIGluIHRoZSBkaW5vZmxhZ2VsbGF0ZSBnZW51cyBMZXBpZG9kaW5pdW0gYXJlIG9mIGNvcmUgY2hsb3JvcGh5dGUgb3JpZ2luOyBQcm90aXN0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTI2OC0yNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGh5bG9nZW5ldGljIG1vc2FpYyBwbGFzdGlkIHByb3Rlb21lIGFuZCB1bnVzdWFsIHBsYXN0aWQtdGFyZ2V0aW5nIHNpZ25hbHMgaW4gdGhlIGdyZWVuLWNvbG9yZWQgZGlub2ZsYWdlbGxhdGUgTGVwaWRvZGluaXVtIGNobG9yb3Bob3J1bTsgQk1DIEV2b2wgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXNzaXZlIGdlbmUgdHJhbnNmZXIgYW5kIGV4dGVuc2l2ZSBSTkEgZWRpdGluZyBvZiBhIHN5bWJpb3RpYyBkaW5vZmxhZ2VsbGF0ZSBwbGFzdGlkIGdlbm9tZTsgR2Vub20gQmlvbCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTQwOC0xNDIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Pcmdhbml6YXRpb24gb2YgdGhlIG1pdG9jaG9uZHJpYWwgZ2Vub21lIGluIHRoZSBkaW5vZmxhZ2VsbGF0ZSBBbXBoaWRpbml1bSBjYXJ0ZXJhZTsgTW9sIEJpb2wgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTI4LTE1MzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCB0cmFuc2NyaXB0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBkaW5vZmxhZ2VsbGF0ZSBwbGFzdGlkIG1pbmljaXJjbGVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5MjsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0yOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBwbGFzdGlkIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgaGFwdG9waHl0ZSBFbWlsaWFuaWEgaHV4bGV5aTogYSBjb21wYXJpc29uIHRvIG90aGVyIHBsYXN0aWQgZ2Vub21lczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNTEtMTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm90YWNoeSBwcm9jZXNzZXMgaW4gcmhvZG9waHl0ZS1kZXJpdmVkIHNlY29uZGhhbmQgcGxhc3RpZCBnZW5lczogaW1wbGljYXRpb25zIGZvciBhZGRyZXNzaW5nIHRoZSBvcmlnaW4gYW5kIGV2b2x1dGlvbiBvZiBkaW5vZmxhZ2VsbGF0ZSBwbGFzdGlkczsgTW9sIEJpb2wgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNTA0LTE1MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcmlhbCByZXBsYWNlbWVudCBvZiBkaWF0b20gZW5kb3N5bWJpb250cyBpbiB0d28gZnJlc2h3YXRlciBkaW5vZmxhZ2VuYXRlcywgUGVyaWRpbmlvcHNpcyBzcHAuIChQZXJpZGluaWFsZXMsIERpbm9waHljZWFlKTsgUGh5Y29sb2dpYTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTQxLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW5zIG9mIHBsYXN0aWRzIGFuZCBnbHljZXJhbGRlaHlkZS0zLXBob3NwaGF0ZSBkZWh5ZHJvZ2VuYXNlIGdlbmVzIGluIHRoZSBncmVlbi1jb2xvcmVkIGRpbm9mbGFnZWxsYXRlIExlcGlkb2Rpbml1bSBjaGxvcm9waG9ydW07IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDEwOyAgY2l0YXRpb25fcGFnZXM9MjYtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWxpbWluYXJ5IHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBwbGFzdGlkLWVuY29kZWQgZ2VuZXMgZnJvbSBhbiBhbm9tYWxvdXNseSBwaWdtZW50ZWQgZGlub2ZsYWdlbGxhdGUgR3ltbm9kaW5pdW0gbWlraW1vdG9pIChHeW1ub2RpbmlhbGVzLCBEaW5vcGh5dGEpOyBQaHljb2wgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTI1Ny0yNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdBUjogV2Vic2VydmVyIGZvciBhbGlnbmluZyBzdHJ1Y3R1cmFsIFJOQXM7IE51Y2wgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTc5LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYW1wYW50IHBvbHl1cmlkeWx5bGF0aW9uIG9mIHBsYXN0aWQgZ2VuZSB0cmFuc2NyaXB0cyBpbiB0aGUgZGlub2ZsYWdlbGxhdGUgTGluZ3Vsb2Rpbml1bTsgTnVjbCBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NjEzLTYxOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Vic3RpdHV0aW9uYWwgZWRpdGluZyBvZiB0cmFuc2NyaXB0cyBmcm9tIGdlbmVzIG9mIGN5YW5vYmFjdGVyaWFsIG9yaWdpbiBpbiB0aGUgZGlub2ZsYWdlbGxhdGUgQ2VyYXRpdW0gaG9ycmlkdW07IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTc3OyAgY2l0YXRpb25fcGFnZXM9NTM1LTUzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIGdlbmUgY2lyY2xlcyBpbiBkaW5vZmxhZ2VsbGF0ZSBjaGxvcm9wbGFzdCBnZW5vbWVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NDAwOyAgY2l0YXRpb25fcGFnZXM9MTU1LTE1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWZvbGQgd2ViIHNlcnZlciBmb3IgbnVjbGVpYyBhY2lkIGZvbGRpbmcgYW5kIGh5YnJpZGl6YXRpb24gcHJlZGljdGlvbjsgTnVjbCBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzQwNi0zNDE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vbWJlL2FydGljbGUtcGRmLzMxLzkvMjM3Ni8xMzE3MTQ5OS9tc3UxODkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFBsYXN0aWRzIHV0aWxpemUgYSBjb21wbGV4IGdlbmUgZXhwcmVzc2lvbiBtYWNoaW5lcnksIHdoaWNoIGhhcyBjb2V2b2x2ZWQgd2l0aCB0aGUgdW5kZXJseWluZyBnZW5vbWUgc2VxdWVuY2UuIFJlbGF0aXZlbHksIGxpdHRsZSBpcyBrbm93biBhYm91dCB0aGUgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yOTI1NzgwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgQWxsIEZpZ3VyZXNkZPUi7yw0onD3AYKbwqPMbaGMIknR" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5325" type="hidden">
        <input id="hdnAdDelaySeconds" value="3000" type="hidden">
        <input id="hdnAdConfigurationTop" value="basic" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="basic" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        <div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad1_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Supplement_Ad1_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Supplement_Ad1_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" frameborder="0" height="90" width="970"></iframe></div></div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic-oup-com.insb.bib.cnrs.fr/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">INIST-CNRS <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

INIST-CNRS
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f9%2f2376%2f2925780" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            INIST-CNRS
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

INIST-CNRS
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f9%2f2376%2f2925780" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944601">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944602">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944599">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944603">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944604">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944605">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944606">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944607">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944608">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944609">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944600">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944610">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944611">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944612">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944613">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944614">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944615">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944616">
            <a href="http://www.oxfordjournals.org.insb.bib.cnrs.fr/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944617">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe" class=" journal-logo-container">
            
                <img id="logo-MolecularBiologyandEvolution" class="journal-logo" src="mbe_title188616415.svg" alt="Molecular Biology and Evolution">
            
                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.smbe.org/smbe/" target="" class="society-logo-container">
                                <img id="logo-SocietyforMolecularBiologyandEvolution" class="society-logo" src="mbe_h1-1277340080.svg" alt="Society for Molecular Biology and Evolution">
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944601">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944602">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944599">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944603">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944604">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944605">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944606">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944607">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944608">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944609">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-944600">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944610">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944611">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944612">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944613">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944614">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944615">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944616">
            <a href="http://www.oxfordjournals.org.insb.bib.cnrs.fr/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-944617">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advanced-search" class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="SMBE Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/smbe" type="hidden">
    <input name="siteID" id="hfsiteID" value="5325" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5538" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic-oup-com.insb.bib.cnrs.fr/mbe" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="mbe" type="hidden">
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    

    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/issue/31/9"><img id="issueImage" class="fb-featured-image" src="m_mbe31_9.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/issue/31/9">
                        <div class="volume trailing-comma">Volume 31</div>
                        <div class="issue">Issue 9</div>

            </a>

        </div>
        <div class="ii-pub-date">
September 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="74404061">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404061">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="74404063">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404063">Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="74404068">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404068">Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="74404105">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404105">Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="74404112">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404112">Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="74404127">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404127">Acknowledgments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="74404129">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#74404129">References</a>
                </div>
            </li>
    <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article/31/9/2365/2925775">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article/31/9/2387/2925782">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide
 Transcript Profiling Reveals the Coevolution of Plastid Gene Sequences 
and Transcript Processing Pathways in the Fucoxanthin Dinoflagellate <em>Karlodinium veneficum</em>
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Elisabeth Richardson</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elisabeth Richardson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/search-results?f_Authors=Elisabeth+Richardson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed?cmd=search&amp;term=Richardson%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com.insb.bib.cnrs.fr/scholar?q=%22author:Richardson%20author:E.%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Richard G. Dorrell</a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Richard G. Dorrell</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/search-results?f_Authors=Richard+G.+Dorrell">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed?cmd=search&amp;term=Dorrell%20R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com.insb.bib.cnrs.fr/scholar?q=%22author:Dorrell%20author:R.%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>
                                <span class="al-author-name">

                                    <a class="linked-name" href="javascript:;">Christopher J. Howe<i class="icon-general-mail"></i></a>
                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christopher J. Howe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Department of Biochemistry, University of Cambridge, Cambridge, United Kingdom</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="msu189-COR1" data-open="msu189-COR1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <strong>*Corresponding author:</strong> E-mail: <a href="mailto:ch26@cam.ac.uk" target="">ch26@cam.ac.uk</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/search-results?f_Authors=Christopher+J.+Howe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed?cmd=search&amp;term=Howe%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com.insb.bib.cnrs.fr/scholar?q=%22author:Howe%20author:C.%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="msu189-FN1"><div class="fn"><div class="footnote-content"><sup>†</sup>These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="msu189-FN2"><div class="fn"><div class="footnote-content"><strong>Associate editor:</strong> Michael Purugganan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="msu189-FN3"><div class="fn"><div class="footnote-content">This
 project has been deposited at GenBank under the accessions 
KF113369-113441, K135651-135653, KM062161-KM062180, and 
KM065532-KM065533.</div></div></div>
    </div>
                                    </div>

                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Molecular Biology and Evolution</em>, Volume 31, Issue 9, 1 September 2014, Pages 2376–2386, <a href="https://doi-org.insb.bib.cnrs.fr/10.1093/molbev/msu189">https://doi-org.insb.bib.cnrs.fr/10.1093/molbev/msu189</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">12 June 2014</div>
                        </div>
                    </div>
                            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
 
                <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2925780" href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/article-pdf/31/9/2376/13171499/msu189.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Elisabeth Richardson, Richard G. Dorrell, Christopher J. Howe;  
Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid Gene
 Sequences and Transcript Processing Pathways in the Fucoxanthin 
Dinoflagellate <em>Karlodinium veneficum</em>, <em>Molecular Biology and Evolution</em>, Volume 31, Issue 9, 1 September 2014, Pages 2376–2386, <a href="https://doi-org.insb.bib.cnrs.fr/10.1093/molbev/msu189">https://doi-org.insb.bib.cnrs.fr/10.1093/molbev/msu189</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=ris">RIS (Zotero)</a></li>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=ris">EndNote</a></li>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=bibtex">BibTex</a></li>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=txt">Medlars</a></li>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=ris">ProCite</a></li>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=txt">RefWorks</a></li>
        <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/downloadcitation/2925780?format=ris">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2017 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1537-1719&amp;title=Genome-Wide%20Transcript%20Profiling%20Reveals%20the%20Coevolution%20of%20Plastid%20Gene%20Sequences%20and%20Transcript%20Processing%20Pathways%20in%20the%20Fucoxanthin%20Dinoflagellate%20%26lt%3Bem%26gt%3BKarlodinium%20veneficum%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2014-06-12&amp;volumeNum=31&amp;issueNum=9&amp;author=Richardson%2C%20Elisabeth%3B%20Dorrell%2C%20Richard%20G.&amp;startPage=2376&amp;endPage=2386&amp;contentId=10.1093%2Fmolbev%2Fmsu189&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  </option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/register?siteId=0&amp;returnUrl=%2fmbe%2farticle%2f31%2f9%2f2376%2f2925780" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  </option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a class="advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="" type="hidden">
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>

                <div class="article-body">
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="74404061" id="74404061" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Plastids utilize a complex gene expression 
machinery, which has coevolved with the underlying genome sequence. 
Relatively, little is known about the genome-wide evolution of 
transcript processing in algal plastids that have undergone complex 
endosymbiotic events. We present the first genome-wide study of 
transcript processing in a plastid acquired through serial 
endosymbiosis, in the fucoxanthin-containing dinoflagellate <em>Karlodinium veneficum</em>.
 The fucoxanthin dinoflagellate plastid has an extremely divergent 
genome and utilizes two unusual transcript processing pathways, 
3′-poly(U) tail addition and sequence editing, which were acquired 
following the serial endosymbiosis event. We demonstrate that poly(U) 
addition and sequence editing are widespread features across the <em>Karl. veneficum</em>
 plastid transcriptome, whereas other dinoflagellate plastid lineages 
that have arisen through independent serial endosymbiosis events do not 
utilize either RNA processing pathway. These pathways constrain the 
effects of divergent sequence evolution in fucoxanthin plastids, for 
example by correcting mutations in the genomic sequence that would 
otherwise be deleterious, and are specifically associated with 
transcripts that encode functional plastid proteins over transcripts of 
recently generated pseudogenes. These pathways may have additionally 
facilitated divergent evolution within the <em>Karl. veneficum</em> plastid. Transcript editing, for example, has contributed to the evolution of a novel C-terminal sequence extension on the <em>Karl. veneficum</em>
 AtpA protein. We furthermore provide the first complete sequence of an 
episomal minicircle in a fucoxanthin dinoflagellate plastid, which 
contains the <em>dnaK</em> gene, and gives rise to polyuridylylated and 
edited transcripts. Our results indicate that RNA processing in 
fucoxanthin dinoflagellate plastids is evolutionarily dynamic, 
coevolving with the underlying genome sequence.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="dinoflagellate">dinoflagellate</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chloroplast">chloroplast</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA processing&quot;">RNA processing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="editing">editing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="polyuridylylation">polyuridylylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="minicircle">minicircle</a></div><div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/search-results?f_TocHeadingTitle=Discoveries">Discoveries</a>
        </div></div>
                            <h2 scrollto-destination="74404063" id="74404063" class="section-title">Introduction</h2>
<p>Plastid gene expression involves a complex set of transcriptional and
 posttranscriptional events. Some of the features of plastid gene 
expression, such as the use of a bacterial RNA polymerase and transcript
 cleavage, are likely to occur universally across the photosynthetic 
eukaryotes (<a reveal-id="msu189-B14" data-open="msu189-B14" class="link link-ref link-reveal xref-bibr">Green 2011</a>).
 Others, such as transcript splicing, sequence editing, and 3′-tail 
addition, appear to have evolved independently within individual plastid
 lineages (<a reveal-id="msu189-B1" data-open="msu189-B1" class="link link-ref link-reveal xref-bibr">Asakura et al. 2008</a>; <a reveal-id="msu189-B22" data-open="msu189-B22" class="link link-ref link-reveal xref-bibr">Lange et al. 2009</a>; <a reveal-id="msu189-B12" data-open="msu189-B12" class="link link-ref link-reveal xref-bibr">Fujii and Small 2011</a>),
 and this may be related to the evolution of the underlying genome 
sequence. For example, transcript editing in plant plastids, which is 
predominantly involved in cytosine deamination, is believed to have 
coevolved with an enrichment in the GC-content of the underlying genome 
sequence relative to the plastids of related green algae (<a reveal-id="msu189-B12" data-open="msu189-B12" class="link link-ref link-reveal xref-bibr">Fujii and Small 2011</a>).</p><p>Until
 recently, very little was known about the evolution of plastid 
transcript processing in lineages other than plants. Some of the most 
important emerging models for studying plastid gene expression in algae 
are dinoflagellates, and their closest relatives, such as the chromerid 
species <em>Chromera velia</em> and <em>Vitrella brassicaformis</em> (<a reveal-id="msu189-B16" data-open="msu189-B16" class="link link-ref link-reveal xref-bibr">Howe et al. 2008</a>; <a reveal-id="msu189-B21" data-open="msu189-B21" class="link link-ref link-reveal xref-bibr">Janouskovec et al. 2013</a>; <a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>).
 Dinoflagellates are an evolutionarily diverse group of algae and 
nonphotosynthetic protists, and have important roles as free-living 
primary producers, and as symbionts of marine invertebrates such as 
coral (<a reveal-id="msu189-B16" data-open="msu189-B16" class="link link-ref link-reveal xref-bibr">Howe et al. 2008</a>).
 The ancestors of all extant dinoflagellates possessed a plastid of red 
algal origin, of the same endosymbiotic derivation as the plastids found
 in chromerids, which is retained in species that contain the pigment 
peridinin (<a reveal-id="msu189-B31" data-open="msu189-B31" class="link link-ref link-reveal xref-bibr">Shalchian-Tabrizi et al. 2006</a>; <a reveal-id="msu189-B20" data-open="msu189-B20" class="link link-ref link-reveal xref-bibr">Janouskovec et al. 2010</a>).
 The peridinin dinoflagellate plastid has an extremely reduced genome, 
containing fewer than 15 genes, many of which are highly divergent in 
sequence, and are located on small, plasmid-like elements termed 
“minicircles” (<a reveal-id="msu189-B38" data-open="msu189-B38" class="link link-ref link-reveal xref-bibr">Zhang et al. 1999</a>; <a reveal-id="msu189-B16" data-open="msu189-B16" class="link link-ref link-reveal xref-bibr">Howe et al. 2008</a>; <a reveal-id="msu189-B14" data-open="msu189-B14" class="link link-ref link-reveal xref-bibr">Green 2011</a>).
 Some dinoflagellates have replaced the peridinin-containing plastids 
with others of a different phylogenetic derivation, through serial 
endosymbiosis. For example, the fucoxanthin-containing dinoflagellates 
possess serially acquired plastids derived from haptophyte algae (<a reveal-id="msu189-B34" data-open="msu189-B34" class="link link-ref link-reveal xref-bibr">Takishita et al. 1999</a>; <a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>; <a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. A near-complete plastid genome sequence has been determined for the fucoxanthin dinoflagellate <em>Karlodinium veneficum</em>, which retains fewer genes than the plastids of free-living haptophytes, and is highly divergent in sequence (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>; <a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>). Other serial endosymbiosis events have occurred in the dinoflagellate genus <em>Lepidodinium</em>, which possesses green algal plastids (<a reveal-id="msu189-B33" data-open="msu189-B33" class="link link-ref link-reveal xref-bibr">Takishita et al. 2008</a>; <a reveal-id="msu189-B26" data-open="msu189-B26" class="link link-ref link-reveal xref-bibr">Matsumoto et al. 2011</a>), and the “dinotom” algae, which possess plastids derived from diatoms (<a reveal-id="msu189-B32" data-open="msu189-B32" class="link link-ref link-reveal xref-bibr">Takano et al. 2008</a>; <a reveal-id="msu189-B18" data-open="msu189-B18" class="link link-ref link-reveal xref-bibr">Imanian et al. 2010</a>). Plastid genome sequences have been assembled for the dinotom species <em>Kryptoperidinium foliaceum</em> and <em>Durinskia baltica</em>, and these retain far more genes, and are less divergent in content than the <em>Karl. veneficum</em> plastid genome (<a reveal-id="msu189-B18" data-open="msu189-B18" class="link link-ref link-reveal xref-bibr">Imanian et al. 2010</a>; <a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>).</p><p>In
 addition to possessing very unusual genomes, dinoflagellate plastids 
utilize a distinctive set of transcript processing pathways. Peridinin 
dinoflagellate plastid transcripts receive a 3′-terminal poly(U) tail, 
and this process, while also found in the plastids of chromerid algae, 
is absent from other plastid lineages, including those of haptophytes 
and diatoms (<a reveal-id="msu189-B36" data-open="msu189-B36" class="link link-ref link-reveal xref-bibr">Wang and Morse 2006</a>; <a reveal-id="msu189-B20" data-open="msu189-B20" class="link link-ref link-reveal xref-bibr">Janouskovec et al. 2010</a>; <a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>.
 In addition, plastid transcripts in some peridinin dinoflagellates 
undergo substitutional sequence editing, which can occur on up to one in
 ten residues in individual transcript sequences and has evolved 
independently from the much less extensive substitutional editing 
observed in land plant plastids (<a reveal-id="msu189-B37" data-open="msu189-B37" class="link link-ref link-reveal xref-bibr">Zauner et al. 2004</a>; <a reveal-id="msu189-B12" data-open="msu189-B12" class="link link-ref link-reveal xref-bibr">Fujii and Small 2011</a>; <a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>.
 Recently, we have shown that 3′-terminal poly(U) tail addition and 
sequence editing occur in plastids of the fucoxanthin dinoflagellate <em>Karenia mikimotoi</em> (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. Editing has been demonstrated in <em>Karl. veneficum</em>, although poly(U) tails have not yet been reported (<a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>).
 As these pathways are associated with peridinin dinoflagellate plastids
 and are not found in free-living haptophytes, they are likely to be 
remnants of the ancestral peridinin-containing plastid symbiosis, 
applied to the fucoxanthin plastid following its uptake by the 
dinoflagellate host (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>.
 These very recently acquired transcript processing pathways in the 
highly divergent fucoxanthin dinoflagellate plastid provide a unique 
opportunity to explore the coevolution of plastid genes and gene 
expression pathways.</p><p>We have surveyed the distribution of poly(U) addition and editing sites across the entire published <em>Karl. veneficum</em> plastid genome (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>; <a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>).
 Our study represents the first genome-wide investigation of RNA 
processing in a plastid acquired by serial endosymbiosis. We demonstrate
 that almost every gene in the <em>Karl. veneficum</em> plastid can give
 rise to polyuridylylated and edited transcripts, including genes that 
are not found in the plastid of peridinin dinoflagellates. We 
demonstrate that the serially acquired plastids in <em>Lepidodinium</em>
 and dinotoms do not utilize either of the RNA processing pathways. We 
have additionally identified unusual roles for poly(U) addition and 
editing in highly divergent regions of the <em>Karl. veneficum</em> 
plastid genome. Poly(U) addition may enable the differentiation of 
functional mRNAs from transcripts of pseudogenes that have arisen 
through recent genome rearrangements, and editing is associated with 
fast-evolving sequences and in-frame insertions that have arisen 
recently in fucoxanthin dinoflagellate plastids. In certain cases, these
 pathways may have indirectly contributed to the evolution of highly 
divergent sequences, such as a novel 3′-extension to the <em>atpA</em> 
coding sequence (CDS) that is generated through transcript editing. Most
 significantly, we present the first complete sequence of an episomal 
minicircle in a serially acquired dinoflagellate plastid, which has 
evolved convergently to the minicircles found in peridinin 
dinoflagellate plastids and gives rise to a polyuridylylated and edited <em>dnaK</em>
 transcript. Our data reveal extensive and complex coevolutionary trends
 between the plastid genome sequence and transcript processing machinery
 of fucoxanthin dinoflagellates.</p>                            <h2 scrollto-destination="74404068" id="74404068" class="section-title">Results</h2>
                            <h3 scrollto-destination="74404069" id="74404069" class="section-title">Presence of Poly(U) Tails on <em>Karl. veneficum</em> Plastid Transcripts</h3>
<p>We investigated whether transcripts in the <em>Karl. veneficum</em> plastid receive 3′-poly(U) tails, as in the related fucoxanthin dinoflagellate <em>Kare. mikimotoi</em> (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. As described previously, we performed reverse transcriptions of <em>Karl. veneficum</em>
 total cellular RNA using an oligo-d(A) primer. We then performed PCR 
using the same oligo-d(A) primer as the PCR reverse primer, and forward 
primers specific to a representative selection of genes across the <em>Karl. veneficum</em> plastid genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>). These included five photosynthesis genes (<em>psbA, psbC, psbD, psaA</em>, and <em>rbcL</em>) previously shown to contain poly(U) sites in <em>Kare. mikimotoi</em> (<a reveal-id="msu189-F1" data-open="msu189-F1" class="link link-reveal link-table xref-fig">fig. 1</a>, lanes 1–5) (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. We additionally tested two plastid housekeeping genes (<em>rpl6</em> and <em>rps5</em>) that have not been investigated in <em>Kare. mikimotoi</em> and are not present in peridinin plastid genomes (<a reveal-id="msu189-F1" data-open="msu189-F1" class="link link-reveal link-table xref-fig">fig. 1</a>, lanes 6 and 7), and a 603-bp open reading frame (ORF) located in a 1,636-bp previously unannotated region between the <em>Karl. veneficum chlI</em> and <em>psbL</em> genes that shows no homology to any previously annotated nucleotide or protein sequence, which we henceforth term <em>ORF1</em> (<a reveal-id="msu189-F1" data-open="msu189-F1" class="link link-reveal link-table xref-fig">fig. 1</a>, lane 8) (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>).
 For each gene tested, we obtained products with the reverse 
transcriptase polymerase chain reaction (RT-PCR). These products were 
sequenced and found to correspond to transcripts containing poly(U) 
sequences located within the 3′-UTR of the gene concerned. These 
sequences did not correspond to poly(T) tracts in the <em>Karl. veneficum</em>
 plastid genome, and hence are posttranscriptional modifications to the 
transcript sequence. Our data thus suggest that a wide variety of 
transcripts in the <em>Karl. veneficum</em> plastid receive poly(U) tails, including transcripts of genes that are not found in the plastids of peridinin dinoflagellates. </p>                        <a id="74404071" scrollto-destination="74404071"></a>
<div id="msu189-F1" data-id="msu189-F1" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f1p.jpg" alt="Presence of poly(U) tails in Karlodinium veneficum plastid transcripts. The gel photo shows the result of a series of representative oligo-d(A) RT-PCRs for specific transcripts from the Karl. veneficum plastid genome. Lanes 1–5: oligo-d(A) RT-PCRs of transcripts that have previously been shown to receive poly(U) tails in Karenia mikimotoi (psbA, psbC, psbD, psaA, rbcL). Lanes 6 and 7: oligo-d(A) RT-PCRs of representative housekeeping genes (rpl6, rps5). Lane 8: oligo-d(A) RT-PCR of the previously unannotated ORF1. Lane 9: RT-PCR of Karl. veneficum psbA using a cDNA template generated using an internal gene-specific cDNA synthesis and PCR reverse primer, and the same psbA forward primer used in lane 1. Lane 10: PCR using the same primers as lane 9, under template negative conditions. The faint secondary band at approximately 1,000 bp in lane 6 corresponds to a dicistronic polyuridylylated rpl6–rps5 transcript. The secondary bands visible in lanes 5, 8, and 9 were found to be PCR chimeras." path-from-xml="msu189f1p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404071/msu189f1p.jpeg" path-from-xml="msu189f1p.jpeg" target="_blank">View large</a><a section="74404071" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f1p.jpeg?Expires=1513001585&amp;Signature=aOw-l1Cc3rcmZwqohmeA5gIxnBd7%7ERM%7ErEXzF5gbQbo0wD4Dbo6wyQ9cc8DwR-e0zC%7EKFHr4YZB-9sJY9czYCrMUclYPvN%7Ec8Wq5oVmZrsXpIGkjPAJMgs%7EvWstD4i6N-Ua77cb6myMiZNXueUdrnmfzboBsIrUP9Z1yhYLMzRCOGzXSGWyUWrR19hz6G2LXTG7%7EwWIvc9AwiTsvSmJWYeGS%7En8oXMFU0HD8pUZu4wN0qGDnsISLg5pKODhPFOyAQiTrtN7KcmFLiS8axwmN9tB7uXcfPKtpdaSI9goHneO4fWScajwDH3YlV3JhT6cDlNAW85XEoge0CfuK4LV-pg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404071&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f1p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Presence of poly(U) tails in <em>Karlodinium veneficum</em>
 plastid transcripts. The gel photo shows the result of a series of 
representative oligo-d(A) RT-PCRs for specific transcripts from the <em>Karl. veneficum</em> plastid genome. Lanes 1–5: oligo-d(A) RT-PCRs of transcripts that have previously been shown to receive poly(U) tails in <em>Karenia mikimotoi</em> (<em>psbA, psbC, psbD, psaA, rbcL</em>). Lanes 6 and 7: oligo-d(A) RT-PCRs of representative housekeeping genes (<em>rpl6, rps5</em>). Lane 8: oligo-d(A) RT-PCR of the previously unannotated <em>ORF1</em>. Lane 9: RT-PCR of <em>Karl. veneficum psbA</em> using a cDNA template generated using an internal gene-specific cDNA synthesis and PCR reverse primer, and the same <em>psbA</em>
 forward primer used in lane 1. Lane 10: PCR using the same primers as 
lane 9, under template negative conditions. The faint secondary band at 
approximately 1,000 bp in lane 6 corresponds to a dicistronic 
polyuridylylated <em>rpl6–rps5</em> transcript. The secondary bands visible in lanes 5, 8, and 9 were found to be PCR chimeras.</p></div></div><div content-id="msu189-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">F<span class="small-caps">ig</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f1p.jpg" alt="Presence of poly(U) tails in Karlodinium veneficum plastid transcripts. The gel photo shows the result of a series of representative oligo-d(A) RT-PCRs for specific transcripts from the Karl. veneficum plastid genome. Lanes 1–5: oligo-d(A) RT-PCRs of transcripts that have previously been shown to receive poly(U) tails in Karenia mikimotoi (psbA, psbC, psbD, psaA, rbcL). Lanes 6 and 7: oligo-d(A) RT-PCRs of representative housekeeping genes (rpl6, rps5). Lane 8: oligo-d(A) RT-PCR of the previously unannotated ORF1. Lane 9: RT-PCR of Karl. veneficum psbA using a cDNA template generated using an internal gene-specific cDNA synthesis and PCR reverse primer, and the same psbA forward primer used in lane 1. Lane 10: PCR using the same primers as lane 9, under template negative conditions. The faint secondary band at approximately 1,000 bp in lane 6 corresponds to a dicistronic polyuridylylated rpl6–rps5 transcript. The secondary bands visible in lanes 5, 8, and 9 were found to be PCR chimeras." path-from-xml="msu189f1p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404071/msu189f1p.jpeg" path-from-xml="msu189f1p.jpeg" target="_blank">View large</a><a section="74404071" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f1p.jpeg?Expires=1513001585&amp;Signature=aOw-l1Cc3rcmZwqohmeA5gIxnBd7%7ERM%7ErEXzF5gbQbo0wD4Dbo6wyQ9cc8DwR-e0zC%7EKFHr4YZB-9sJY9czYCrMUclYPvN%7Ec8Wq5oVmZrsXpIGkjPAJMgs%7EvWstD4i6N-Ua77cb6myMiZNXueUdrnmfzboBsIrUP9Z1yhYLMzRCOGzXSGWyUWrR19hz6G2LXTG7%7EwWIvc9AwiTsvSmJWYeGS%7En8oXMFU0HD8pUZu4wN0qGDnsISLg5pKODhPFOyAQiTrtN7KcmFLiS8axwmN9tB7uXcfPKtpdaSI9goHneO4fWScajwDH3YlV3JhT6cDlNAW85XEoge0CfuK4LV-pg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404071&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f1p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Presence of poly(U) tails in <em>Karlodinium veneficum</em>
 plastid transcripts. The gel photo shows the result of a series of 
representative oligo-d(A) RT-PCRs for specific transcripts from the <em>Karl. veneficum</em> plastid genome. Lanes 1–5: oligo-d(A) RT-PCRs of transcripts that have previously been shown to receive poly(U) tails in <em>Karenia mikimotoi</em> (<em>psbA, psbC, psbD, psaA, rbcL</em>). Lanes 6 and 7: oligo-d(A) RT-PCRs of representative housekeeping genes (<em>rpl6, rps5</em>). Lane 8: oligo-d(A) RT-PCR of the previously unannotated <em>ORF1</em>. Lane 9: RT-PCR of <em>Karl. veneficum psbA</em> using a cDNA template generated using an internal gene-specific cDNA synthesis and PCR reverse primer, and the same <em>psbA</em>
 forward primer used in lane 1. Lane 10: PCR using the same primers as 
lane 9, under template negative conditions. The faint secondary band at 
approximately 1,000 bp in lane 6 corresponds to a dicistronic 
polyuridylylated <em>rpl6–rps5</em> transcript. The secondary bands visible in lanes 5, 8, and 9 were found to be PCR chimeras.</p></div></div><p>To
 confirm that the oligo-d(A) primed RT-PCR products correspond to 
3′-terminal transcript poly(U) tails, as opposed to internal sequence 
insertions, or to artifacts generated by mispriming of the oligo-d(A) 
primer, we performed RT-PCRs using circular RNA and cDNA and PCR 
synthesis primers specific to the <em>Karl. veneficum psbA</em> and <em>psbC</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). We have previously employed this technique successfully to confirm the presence of polyuridylylated <em>psbA</em> and <em>psbC</em> transcripts in <em>Kare. mikimotoi</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. We identified 3′-terminal poly(U) tails on the ends of <em>Karl. veneficum psbA</em> and <em>psbC</em> transcripts using this approach (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Although we additionally identified nonpolyuridylylated <em>psbA</em>
 transcripts, all of these transcripts terminated at the 3′-end within 
the CDS and are therefore likely to represent transcript degradation 
products as opposed to mature transcripts generated by a 
poly(U)-independent processing pathway (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Our data confirm that poly(U) tails are added to a wide variety of plastid transcripts in <em>Karl. veneficum</em>, as with <em>Kare. mikimotoi</em>, and suggest that the poly(U) addition pathway was acquired by a common ancestor of extant fucoxanthin dinoflagellates.</p>                            <h3 scrollto-destination="74404073" id="74404073" class="section-title">Extent of Poly(U) Addition within the <em>Karl. veneficum</em> Plastid</h3>
<p>We extended the initial analysis to determine the total extent of transcript polyuridylylation in the <em>Karl. veneficum</em>
 plastid. We performed oligo-d(A) primed RT-PCRs using PCR forward 
primers for every annotated protein-coding and ribosomal RNA gene within
 the plastid genome, including previously unannotated <em>atpE, petG,</em> and <em>rps10</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary tables S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Data</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). We also tested for the presence of poly(U) tails for 15 predicted tRNA genes in the <em>Karl. veneficum</em>
 plastid genome, and three further predicted ORFs of more than 300 bp 
length that bear no sequence homology to any previously identified 
plastid gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>). We found evidence for widespread polyuridylylation of the <em>Karl. veneficum</em>
 plastid transcriptome, with 54 of the 75 protein-coding genes, and two 
of the four novel ORFs surveyed possessing poly(U) sites in the 
associated 3′-UTR (<a reveal-id="msu189-F2" data-open="msu189-F2" class="link link-reveal link-table xref-fig">fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Four of the 56 poly(U) sites observed were positioned within genomic poly(T) tracts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online), so it is possible that they have arisen through primer 
misannealing. However, the remaining 52 were not and are likely to 
correspond to posttranscriptional modifications. </p>                        <a id="74404075" scrollto-destination="74404075"></a>
<div id="msu189-F2" data-id="msu189-F2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f2p.jpg" alt="Extent of transcript polyuridylylation across the Karlodinium veneficum plastid. The Venn diagram shows the transcript polyuridylylation state of every gene within the Karl. veneficum plastid genome. Genes in the overlap sector between the two circles lack directly associated poly(U) sites in their respective 3′-UTR sequences, but can be retrieved as part of polyuridylylated polycistronic transcripts, with the poly(U) site positioned in the 3′-UTR of a downstream gene. The poly(U) tails of genes shaded in gray may be generated by the transcription of genomic poly(T) tracts." path-from-xml="msu189f2p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404075/msu189f2p.jpeg" path-from-xml="msu189f2p.jpeg" target="_blank">View large</a><a section="74404075" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f2p.jpeg?Expires=1513001585&amp;Signature=I6VUH3RL3pSdm-Xp16FVm2L6YzJspqmPOQcgQV4V5%7EtI02zNhykJ64GWyqzHoNzyw4sZgYzr%7EFGwWnTqyays9iODfM4KEOH74onVpbEmawoLkp85dlFaWzQgdE8%7EdRW4DahAmNVidMgONlZkDMRgDxb3hFb%7EpRlkTEyshyYsllAP%7En9E7nTGgoLFcp8cJ4MpueM2okhFIdjKcZCSsiIBgenyN6u-8nH6J5nvy2vRQSLzuLAiy-74eZGxOVKvRS5kuNVg-%7EBmGxs-mwISgGkKNMOJj9%7EtRzXdQDHQB2O3CG9BhWzlMuEcRiFvC0c4gcbqiVDZXZZjilG7EqmZGsKFZQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404075&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f2p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Extent of transcript polyuridylylation across the <em>Karlodinium veneficum</em> plastid. The Venn diagram shows the transcript polyuridylylation state of every gene within the <em>Karl. veneficum</em>
 plastid genome. Genes in the overlap sector between the two circles 
lack directly associated poly(U) sites in their respective 3′-UTR 
sequences, but can be retrieved as part of polyuridylylated 
polycistronic transcripts, with the poly(U) site positioned in the 
3′-UTR of a downstream gene. The poly(U) tails of genes shaded in gray 
may be generated by the transcription of genomic poly(T) tracts.</p></div></div><div content-id="msu189-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f2p.jpg" alt="Extent of transcript polyuridylylation across the Karlodinium veneficum plastid. The Venn diagram shows the transcript polyuridylylation state of every gene within the Karl. veneficum plastid genome. Genes in the overlap sector between the two circles lack directly associated poly(U) sites in their respective 3′-UTR sequences, but can be retrieved as part of polyuridylylated polycistronic transcripts, with the poly(U) site positioned in the 3′-UTR of a downstream gene. The poly(U) tails of genes shaded in gray may be generated by the transcription of genomic poly(T) tracts." path-from-xml="msu189f2p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404075/msu189f2p.jpeg" path-from-xml="msu189f2p.jpeg" target="_blank">View large</a><a section="74404075" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f2p.jpeg?Expires=1513001585&amp;Signature=I6VUH3RL3pSdm-Xp16FVm2L6YzJspqmPOQcgQV4V5%7EtI02zNhykJ64GWyqzHoNzyw4sZgYzr%7EFGwWnTqyays9iODfM4KEOH74onVpbEmawoLkp85dlFaWzQgdE8%7EdRW4DahAmNVidMgONlZkDMRgDxb3hFb%7EpRlkTEyshyYsllAP%7En9E7nTGgoLFcp8cJ4MpueM2okhFIdjKcZCSsiIBgenyN6u-8nH6J5nvy2vRQSLzuLAiy-74eZGxOVKvRS5kuNVg-%7EBmGxs-mwISgGkKNMOJj9%7EtRzXdQDHQB2O3CG9BhWzlMuEcRiFvC0c4gcbqiVDZXZZjilG7EqmZGsKFZQ__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404075&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f2p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Extent of transcript polyuridylylation across the <em>Karlodinium veneficum</em> plastid. The Venn diagram shows the transcript polyuridylylation state of every gene within the <em>Karl. veneficum</em>
 plastid genome. Genes in the overlap sector between the two circles 
lack directly associated poly(U) sites in their respective 3′-UTR 
sequences, but can be retrieved as part of polyuridylylated 
polycistronic transcripts, with the poly(U) site positioned in the 
3′-UTR of a downstream gene. The poly(U) tails of genes shaded in gray 
may be generated by the transcription of genomic poly(T) tracts.</p></div></div><p>For
 some of the oligo-d(A) RT-PCRs we identified multiple products, 
consistent with the presence of different polyuridylylated transcripts 
from a single gene. For example, in the case of <em>rpl6</em>, in 
addition to obtaining PCR products of a size consistent with a 
monocistronic, polyuridylylated transcript, we observed a secondary 
higher molecular weight product that was found to correspond to a 
polyuridylylated dicistronic <em>rpl6</em><em>–rps5</em> transcript (<a reveal-id="msu189-F1" data-open="msu189-F1" class="link link-reveal link-table xref-fig">fig. 1</a>,
 lane 7). We additionally obtained polyuridylylated dicistronic 
transcripts for 13 of the 21 protein-coding genes that lacked poly(U) 
sites immediately downstream but were positioned directly upstream of 
genes that possessed poly(U) sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online, and <a reveal-id="msu189-F2" data-open="msu189-F2" class="link link-reveal link-table xref-fig">fig. 2</a>).
 This indicates that even genes that do not possess directly adjacent 
poly(U) sites may give rise to polyuridylylated transcripts.</p><p>A small number of the protein-coding genes and unannotated ORFs in the <em>Karl. veneficum</em> plastid failed to yield significant products in any oligo-d(A) RT-PCR attempted (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online, and <a reveal-id="msu189-F2" data-open="msu189-F2" class="link link-reveal link-table xref-fig">fig. 2</a>).
 In each case, we failed to detect products for each gene even following
 a nested reamplification of the primary PCR product, with the same 
oligo-d(A) primer and a second gene-specific primer positioned 
downstream of the first (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). None of these genes was positioned directly upstream of a gene
 in the same transcriptional orientation that possessed a poly(U) site, 
suggesting that they are unlikely to give rise to polycistronic 
polyuridylylated transcripts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Data</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). We amplified transcript sequences for these genes, using cDNA 
synthesis primers internal to the CDS, and these were not completely 
identical to the underlying genomic sequence, consistent with transcript
 editing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary tables S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Data</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). We similarly could not identify products in oligo-d(A) RT-PCRs
 using primers specific to any of the ribosomal RNA subunits or tRNA 
genes although we identified a tricistronic polyuridylylated <em>rrs-petG-atpF-1</em> transcript (<a reveal-id="msu189-F2" data-open="msu189-F2" class="link link-reveal link-table xref-fig">fig.2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). We generated transcript sequences for all three ribosomal 
subunits (5.8S, 16S, and 23S rRNA), and could detect low levels of 
editing in each case (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary tables S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Data</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). Our data indicate that the poly(U) addition and editing 
machinery have been co-opted to recognize almost every gene in the <em>Karl. veneficum</em> plastid.</p>                            <h3 scrollto-destination="74404078" id="74404078" class="section-title">Location of Poly(U) Sites</h3>
<p>We wished to determine what sequence features were associated with the presence of poly(U) sites in the <em>Karl. veneficum</em> plastid genome. In chromerid algae, poly(U) addition is biased toward genes encoding proteins that function in photosynthesis (<a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>). Although photosynthesis genes in the <em>Karl. veneficum</em>
 plastid are more likely to possess an associated poly(U) site than 
housekeeping genes, the association is not statistically significant 
(chi-squared, <em>P</em> = 0.07) (<a reveal-id="msu189-F2" data-open="msu189-F2" class="link link-reveal link-table xref-fig">fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). We additionally compared the gene order of the <em>Karl. veneficum</em>
 genome with those of free-living haptophyte species, and could not 
identify a consistent relationship between the absence of a poly(U) 
site, and inferred recombination events (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S3</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). Our data therefore indicate that gene function and genome 
rearrangements are unlikely to be the only factors that determine the 
distribution of poly(U) sites across the <em>Karl. veneficum</em> plastid.</p><p>The poly(U) sites within the <em>Karl. veneficum</em> plastid are typically positioned close to the 3′-end of the CDS, with an average 3′-UTR length of only 30 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). We looked for conserved primary sequence motifs, changes in GC
 and purine/pyrimidine content, and predicted RNA secondary structures 
in the 3′-UTR sequences of each gene, extending 100 bp downstream of 
each poly(U) site. We could not identify any sequence features that were
 significantly associated with the presence of a poly(U) site. Several 
of the poly(U) sites, however, were located within the CDS of the 
downstream gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Most dramatically, within the ten-gene ribosomal protein operon extending from <em>rpl3</em> through to <em>rps5</em>, we identified four genes (<em>rpl3, rpl16, rps8,</em> and <em>rpl6</em>)
 where the poly(U) site overlaps with the downstream CDS, whereas we 
only found one poly(U) site within a 3′-UTR sequence, associated with <em>rps5</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S3</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>). Using a forward primer specific to <em>rpl2,</em> we additionally detected a poly(U) site located 296 bp within the <em>rpl2</em> CDS although we could not identify this site using a forward primer specific to the upstream <em>rpl3</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S3</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). The poly(U) sites located internal to gene sequences might be 
associated with alternative end processing events as their formation 
would prevent transcripts of specific genes being produced from 
polycistronic precursors. Overall, our data suggest that instead of 
poly(U) addition being associated with common sequence motifs or 
specific genes, poly(U) sites are highly sequence-specific. The 
formation of specific poly(U) sites might influence other events in 
plastid transcript processing.</p>                            <h3 scrollto-destination="74404081" id="74404081" class="section-title">Differential Recognition of Pseudogenes by the <em>Karl. veneficum</em> Plastid Transcript Processing Machinery</h3>
<p>It has been demonstrated that poly(U) addition discriminates between paralogous copies of genes in the <em>C. velia</em> plastid (<a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>). Transcripts of the <em>C. velia atpH-1</em> gene, which are abundant, receive a poly(U) tail, whereas transcripts of the <em>atpH-2</em> gene, which appears to be a pseudogene, do not (<a reveal-id="msu189-B21" data-open="msu189-B21" class="link link-ref link-reveal xref-bibr">Janouskovec et al. 2013</a>; <a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>). Several of the genes in the <em>Karl. veneficum</em> plastid are present in multiple copies, some of which appear to be functional, whereas others are likely to be pseudogenes (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>). For example, two copies of the <em>rbcS</em> gene are present: <em>rbcS-2</em>, which is likely to encode a functional protein, and <em>rbcS-1</em>,
 which contains an in-frame insertion within the region encoding the 
βC-βD loop domain of the rubisco small subunit, that if expressed would 
be likely to interfere with its function (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S4A</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B23" data-open="msu189-B23" class="link link-ref link-reveal xref-bibr">Larson et al. 1997</a>; <a reveal-id="msu189-B24" data-open="msu189-B24" class="link link-ref link-reveal xref-bibr">Li et al. 2005</a>). Similarly, we identified two copies of the <em>atpF</em> gene: a previously annotated gene (<em>atpF-1</em>) and a previously unannotated pseudogene (<em>atpF-2</em>)<em>,</em> positioned downstream of and in reverse orientation to <em>psbB,</em> which contains an internal frame-shift sequence deletion that would prevent the translation of the complete protein sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S4B</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Data</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online).</p><p>We wished to determine whether transcripts of the <em>rbcS-1</em> and <em>atpF-2</em> pseudogenes receive poly(U) tails and are edited. We could detect polyuridylylated <em>rbcS-2</em> and <em>atpF-1</em> transcripts by oligo-d(A) primed RT-PCR, using PCR forward primers specific to each sequence (<a reveal-id="msu189-F3" data-open="msu189-F3" class="link link-reveal link-table xref-fig">fig. 3</a>, lanes 2 and 5), but could not detect polyuridylylated <em>rbcS-1</em> and <em>atpF-2</em> transcripts through the same approach (<a reveal-id="msu189-F3" data-open="msu189-F3" class="link link-reveal link-table xref-fig">fig. 3</a>, lanes 1 and 6). We could amplify nonpolyuridylylated <em>rbcS-1</em> and <em>atpF-2</em> transcript sequences by performing RT-PCRs against cDNA synthesis primers specific to each gene (<a reveal-id="msu189-F3" data-open="msu189-F3" class="link link-reveal link-table xref-fig">fig. 3</a>, lanes 3–4, 7–8). We sequenced the products of these RT-PCRs, and confirmed the presence of the in-frame insertion in <em>rbcS-1</em> and the frame-shift deletion in <em>atpF-2</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). We could not identify any editing within the <em>atpF-2</em> transcript, and detected only one editing event on the <em>rbcS-1</em> transcript, which is significantly fewer than the 15 editing events observed over the same region of the <em>rbcS-2</em> transcript sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online; binomial test, P&lt; E-05). Our data indicate that poly(U) 
addition and editing are preferentially associated with functional genes
 in the <em>Karl. veneficum</em> plastid. </p>                        <a id="74404084" scrollto-destination="74404084"></a>
<div id="msu189-F3" data-id="msu189-F3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f3p.jpg" alt="Specific addition of poly(U) tails to transcripts of functional gene paralogs in the Karlodinium veneficum plastid. This gel photo shows the result of a series of RT-PCRs to identify whether transcripts of functional and pseudogenic copies of the rbcS and atpF genes in the Karl. veneficum plastid receive poly(U) tails. Lanes 1 and 2: oligo-d(A) RT-PCR of rbcS-1 (pseudogene) and rbcS-2 (functional). Lanes 3 and 4: RT-PCR of rbcS-1 with a gene-specific internal cDNA synthesis primer under template positive (lane 3) and negative (lane 4) conditions. Lanes 5 and 6: oligo-d(A) RT-PCR of atpF-1 (highly divergent gene) and the atpF-2 nonfunctional sequence between rps16 and psbB. Lanes 7 and 8: RT-PCR of the atpF-2 region with a gene-specific cDNA synthesis primer under template positive (lane 7) and negative (lane 8) conditions." path-from-xml="msu189f3p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404084/msu189f3p.jpeg" path-from-xml="msu189f3p.jpeg" target="_blank">View large</a><a section="74404084" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f3p.jpeg?Expires=1513001585&amp;Signature=Uf%7EM0mMXtD-fRUHKTAgOrkp-%7Ep%7EPnb1r49E8yB4Utf-WZUd7bJSBbWIOkJIBu8liJTo96PzLsQ-E7jL8eo0B0I3zKHGUv6ou82W8%7EifQq27u6puwq-Hx7q-BF4YH9qG87Iz5Hs7%7E2o7oq85Km60NuMwWIFeJhbn5RCTiBAXhANZ02tCtGT4jMZySqf2z7vkVbqcL-uGoAiVZ3iC5odzlvnWxBiAhh2%7E5J%7E-hcOU261VJLNkujMPzCLiplPERhVDZ7LZkcQLgsZwdLC%7ErmtIZao6yC-5PeclRsLP25knMYSImbeCIGbvrT%7EYYm1z-xVbGIoFo0GVKgVnURPDmRWp3-A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404084&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f3p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Specific addition of poly(U) tails to transcripts of functional gene paralogs in the <em>Karlodinium veneficum</em>
 plastid. This gel photo shows the result of a series of RT-PCRs to 
identify whether transcripts of functional and pseudogenic copies of the
 <em>rbcS</em> and <em>atpF</em> genes in the <em>Karl. veneficum</em> plastid receive poly(U) tails. Lanes 1 and 2: oligo-d(A) RT-PCR of <em>rbcS-1</em> (pseudogene) and <em>rbcS-2</em> (functional). Lanes 3 and 4: RT-PCR of <em>rbcS-1</em>
 with a gene-specific internal cDNA synthesis primer under template 
positive (lane 3) and negative (lane 4) conditions. Lanes 5 and 6: 
oligo-d(A) RT-PCR of <em>atpF-1</em> (highly divergent gene) and the <em>atpF-2</em> nonfunctional sequence between <em>rps16</em> and <em>psbB</em>. Lanes 7 and 8: RT-PCR of the <em>atpF-2</em> region with a gene-specific cDNA synthesis primer under template positive (lane 7) and negative (lane 8) conditions.</p></div></div><div content-id="msu189-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f3p.jpg" alt="Specific addition of poly(U) tails to transcripts of functional gene paralogs in the Karlodinium veneficum plastid. This gel photo shows the result of a series of RT-PCRs to identify whether transcripts of functional and pseudogenic copies of the rbcS and atpF genes in the Karl. veneficum plastid receive poly(U) tails. Lanes 1 and 2: oligo-d(A) RT-PCR of rbcS-1 (pseudogene) and rbcS-2 (functional). Lanes 3 and 4: RT-PCR of rbcS-1 with a gene-specific internal cDNA synthesis primer under template positive (lane 3) and negative (lane 4) conditions. Lanes 5 and 6: oligo-d(A) RT-PCR of atpF-1 (highly divergent gene) and the atpF-2 nonfunctional sequence between rps16 and psbB. Lanes 7 and 8: RT-PCR of the atpF-2 region with a gene-specific cDNA synthesis primer under template positive (lane 7) and negative (lane 8) conditions." path-from-xml="msu189f3p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404084/msu189f3p.jpeg" path-from-xml="msu189f3p.jpeg" target="_blank">View large</a><a section="74404084" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f3p.jpeg?Expires=1513001585&amp;Signature=Uf%7EM0mMXtD-fRUHKTAgOrkp-%7Ep%7EPnb1r49E8yB4Utf-WZUd7bJSBbWIOkJIBu8liJTo96PzLsQ-E7jL8eo0B0I3zKHGUv6ou82W8%7EifQq27u6puwq-Hx7q-BF4YH9qG87Iz5Hs7%7E2o7oq85Km60NuMwWIFeJhbn5RCTiBAXhANZ02tCtGT4jMZySqf2z7vkVbqcL-uGoAiVZ3iC5odzlvnWxBiAhh2%7E5J%7E-hcOU261VJLNkujMPzCLiplPERhVDZ7LZkcQLgsZwdLC%7ErmtIZao6yC-5PeclRsLP25knMYSImbeCIGbvrT%7EYYm1z-xVbGIoFo0GVKgVnURPDmRWp3-A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404084&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f3p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Specific addition of poly(U) tails to transcripts of functional gene paralogs in the <em>Karlodinium veneficum</em>
 plastid. This gel photo shows the result of a series of RT-PCRs to 
identify whether transcripts of functional and pseudogenic copies of the
 <em>rbcS</em> and <em>atpF</em> genes in the <em>Karl. veneficum</em> plastid receive poly(U) tails. Lanes 1 and 2: oligo-d(A) RT-PCR of <em>rbcS-1</em> (pseudogene) and <em>rbcS-2</em> (functional). Lanes 3 and 4: RT-PCR of <em>rbcS-1</em>
 with a gene-specific internal cDNA synthesis primer under template 
positive (lane 3) and negative (lane 4) conditions. Lanes 5 and 6: 
oligo-d(A) RT-PCR of <em>atpF-1</em> (highly divergent gene) and the <em>atpF-2</em> nonfunctional sequence between <em>rps16</em> and <em>psbB</em>. Lanes 7 and 8: RT-PCR of the <em>atpF-2</em> region with a gene-specific cDNA synthesis primer under template positive (lane 7) and negative (lane 8) conditions.</p></div></div>                            <h3 scrollto-destination="74404085" id="74404085" class="section-title">Global Trends in Editing across the <em>Karl. veneficum</em> Plastid Transcriptome</h3>
<p>Recently, <a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. (2013)</a> have profiled editing events in the <em>Karl. veneficum</em>
 plastid by comparing transcript and genomic sequences for regions of 14
 different genes. Four different forms of editing were observed, all of 
which were transitions, consisting predominantly of A to G and U to C 
editing events, as well as small numbers of G to A and C to U 
conversions (<a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>). Across our entire data set, we found evidence for extensive sequence editing (<a reveal-id="msu189-T1" data-open="msu189-T1" class="link link-reveal link-table xref-fig">table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Approximately 4.3% of sites in our transcript sequences were edited, slightly higher than previous estimates (<a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>). For some genes, we detected higher frequencies of editing, extending to 14% of positions for the <em>Karl. veneficum psbD</em> gene, and 24% of residues in the highly divergent <em>petG</em> sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). Editing sites were situated predominantly within gene 
sequences although we detected a low level of editing (1.6%) in 
polyuridylylated transcript 3′-UTR sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online), as previously seen in <em>Kare. mikimotoi</em> (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. Many (88%) of the editing events lead to an increase in transcript GC-content, consistent with previous studies (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012</a>; <a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>) (<a reveal-id="msu189-T1" data-open="msu189-T1" class="link link-reveal link-table xref-fig">table 1</a>).
 Although the majority (96%) of editing events observed were transition 
events, we detected seven different transversion events at low levels in
 the <em>Karl. veneficum</em> transcriptome, similar to our previous observations in <em>Kare. mikimotoi</em> (<a reveal-id="msu189-T1" data-open="msu189-T1" class="link link-reveal link-table xref-fig">table 1</a>) (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. </p>                        <a id="74404087" scrollto-destination="74404087"></a>
<div class="table-wrap"><div id="msu189-T1" class="table-wrap-title" data-id="msu189-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Total Editing Events from the Characterized Plastid Transcriptomes of <em>Karenia mikimotoi</em> and <em>Karlodinium veneficum</em>.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Overview&nbsp;</th><th><em>Karenia</em>&nbsp;</th><th><em>Karlodinium</em>—Jackson&nbsp;</th><th><em>Karlodinium</em>—Extended&nbsp;</th></tr></thead><tbody><tr><td>Sequence length&nbsp;</td><td>5,473&nbsp;</td><td>7,373&nbsp;</td><td>36,084&nbsp;</td></tr><tr><td>A-C&nbsp;</td><td>26&nbsp;</td><td>0&nbsp;</td><td>15&nbsp;</td></tr><tr><td>A-G&nbsp;</td><td>59&nbsp;</td><td>131&nbsp;</td><td>789&nbsp;</td></tr><tr><td>A-U&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>16&nbsp;</td></tr><tr><td>C-A&nbsp;</td><td>1&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>C-G&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>4&nbsp;</td></tr><tr><td>C-U&nbsp;</td><td>17&nbsp;</td><td>8&nbsp;</td><td>49&nbsp;</td></tr><tr><td>G-A&nbsp;</td><td>15&nbsp;</td><td>15&nbsp;</td><td>99&nbsp;</td></tr><tr><td>G-C&nbsp;</td><td>24&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>G-U&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>1&nbsp;</td></tr><tr><td>U-A&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>11&nbsp;</td></tr><tr><td>U-C&nbsp;</td><td>116&nbsp;</td><td>67&nbsp;</td><td>540&nbsp;</td></tr><tr><td>U-G&nbsp;</td><td>2&nbsp;</td><td>0&nbsp;</td><td>11&nbsp;</td></tr><tr><td>Total&nbsp;</td><td>260&nbsp;</td><td>221&nbsp;</td><td>1,539&nbsp;</td></tr><tr><td>% Bases edited&nbsp;</td><td>4.75&nbsp;</td><td>2.86&nbsp;</td><td>4.27&nbsp;</td></tr><tr><td>% Transitions&nbsp;</td><td>79.6&nbsp;</td><td>100&nbsp;</td><td>96&nbsp;</td></tr><tr><td>% Transversions&nbsp;</td><td>20.4&nbsp;</td><td>0&nbsp;</td><td>4&nbsp;</td></tr><tr><td>% GC-enrich&nbsp;</td><td>78.1&nbsp;</td><td>89.6&nbsp;</td><td>88&nbsp;</td></tr><tr><td>% GC-deplete&nbsp;</td><td>12.7&nbsp;</td><td>10.4&nbsp;</td><td>10&nbsp;</td></tr><tr><td>% GC-neutral&nbsp;</td><td>9.2&nbsp;</td><td>0&nbsp;</td><td>1.9&nbsp;</td></tr><tr><td>% Nonsynonymous&nbsp;</td><td>58.5&nbsp;</td><td>95&nbsp;</td><td>87.1&nbsp;</td></tr><tr><td>% Synonymous&nbsp;</td><td>41.5&nbsp;</td><td>5&nbsp;</td><td>12.9&nbsp;</td></tr></tbody></table></div><div content-id="msu189-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Overview&nbsp;</th><th><em>Karenia</em>&nbsp;</th><th><em>Karlodinium</em>—Jackson&nbsp;</th><th><em>Karlodinium</em>—Extended&nbsp;</th></tr></thead><tbody><tr><td>Sequence length&nbsp;</td><td>5,473&nbsp;</td><td>7,373&nbsp;</td><td>36,084&nbsp;</td></tr><tr><td>A-C&nbsp;</td><td>26&nbsp;</td><td>0&nbsp;</td><td>15&nbsp;</td></tr><tr><td>A-G&nbsp;</td><td>59&nbsp;</td><td>131&nbsp;</td><td>789&nbsp;</td></tr><tr><td>A-U&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>16&nbsp;</td></tr><tr><td>C-A&nbsp;</td><td>1&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>C-G&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>4&nbsp;</td></tr><tr><td>C-U&nbsp;</td><td>17&nbsp;</td><td>8&nbsp;</td><td>49&nbsp;</td></tr><tr><td>G-A&nbsp;</td><td>15&nbsp;</td><td>15&nbsp;</td><td>99&nbsp;</td></tr><tr><td>G-C&nbsp;</td><td>24&nbsp;</td><td>0&nbsp;</td><td>8&nbsp;</td></tr><tr><td>G-U&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>1&nbsp;</td></tr><tr><td>U-A&nbsp;</td><td>0&nbsp;</td><td>0&nbsp;</td><td>11&nbsp;</td></tr><tr><td>U-C&nbsp;</td><td>116&nbsp;</td><td>67&nbsp;</td><td>540&nbsp;</td></tr><tr><td>U-G&nbsp;</td><td>2&nbsp;</td><td>0&nbsp;</td><td>11&nbsp;</td></tr><tr><td>Total&nbsp;</td><td>260&nbsp;</td><td>221&nbsp;</td><td>1,539&nbsp;</td></tr><tr><td>% Bases edited&nbsp;</td><td>4.75&nbsp;</td><td>2.86&nbsp;</td><td>4.27&nbsp;</td></tr><tr><td>% Transitions&nbsp;</td><td>79.6&nbsp;</td><td>100&nbsp;</td><td>96&nbsp;</td></tr><tr><td>% Transversions&nbsp;</td><td>20.4&nbsp;</td><td>0&nbsp;</td><td>4&nbsp;</td></tr><tr><td>% GC-enrich&nbsp;</td><td>78.1&nbsp;</td><td>89.6&nbsp;</td><td>88&nbsp;</td></tr><tr><td>% GC-deplete&nbsp;</td><td>12.7&nbsp;</td><td>10.4&nbsp;</td><td>10&nbsp;</td></tr><tr><td>% GC-neutral&nbsp;</td><td>9.2&nbsp;</td><td>0&nbsp;</td><td>1.9&nbsp;</td></tr><tr><td>% Nonsynonymous&nbsp;</td><td>58.5&nbsp;</td><td>95&nbsp;</td><td>87.1&nbsp;</td></tr><tr><td>% Synonymous&nbsp;</td><td>41.5&nbsp;</td><td>5&nbsp;</td><td>12.9&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="msu189-TF1"><div class="fn"><p>N<span class="small-caps">ote</span>.—The total editing events observed across 36,084-bp <em>Karlodinium veneficum</em> plastid transcript sequence in this study are profiled, alongside previous surveys of <em>Karl. veneficum</em> (<a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>), and of the related fucoxanthin dinoflagellate <em>Kare. mikimotoi</em> (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/74404087" target="_blank">View Large</a></div></div><p>Most (87%) of the editing events in the <em>Karl. veneficum</em> plastid are predicted to have nonsynonymous effects on the corresponding protein sequence (<a reveal-id="msu189-T1" data-open="msu189-T1" class="link link-reveal link-table xref-fig">table 1</a>).
 Some of these editing events may be required for the correct function 
of the encoded protein. For example, 11 of the genes in the <em>Karl. veneficum</em>
 plastid contain premature in-frame termination codons, which would 
prevent the translation of the complete protein sequence. Correction of 
premature termination codons through editing has previously been 
reported for <em>Karl. veneficum rpoB, rps13, psaA</em> and <em>secY</em> transcripts, and <em>psaA</em> in <em>Kare. mikimotoi</em> (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012</a>; <a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>). We confirmed that all of the premature termination codons in the <em>Karl. veneficum</em> genome are removed from the corresponding polyuridylylated transcript sequences by editing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Consistent with previous reports, we also found that edited <em>Karl. veneficum</em>
 transcripts show an increase in sequence similarity, relative to the 
genomic sequence, to the corresponding sequences from the haptophytes <em>Emiliania huxleyi</em> and <em>Phaeocystis globosa</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>). Editing in the <em>Karl. veneficum</em> plastid therefore appears to reduce the effects of divergent mutations on plastid protein sequence.</p>                            <h3 scrollto-destination="74404089" id="74404089" class="section-title">Editing of Fast-Evolving Sequences in the <em>Karl. veneficum</em> Plastid</h3>
<p>Not all of the nonsynonymous editing events observed within the <em>Karl. veneficum</em>
 plastid have readily inferred effects on plastid protein function. 
Across our entire data set, we found that although more than one in ten 
codons undergo a nonsynonymous change due to editing, this only leads to
 a net increase of 1.6% in sequence conservation between the <em>Karl. veneficum</em> and haptophyte protein sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). The other editing events may have selectively neutral or 
disadvantageous effects, or affect sequences that are not found in 
free-living haptophytes. Notably, many of the genes in the <em>Karl. veneficum</em> plastid genome contain novel sequence insertions, or fast-diverging regions that bear no homology to haptophyte sequences (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>).
 We hypothesized that editing events that do not increase sequence 
conservation with haptophyte orthologs might instead affect sequences 
unique to the <em>Karl. veneficum</em> plastid.</p><p>Certain transcripts within our data set contain highly edited regions. For example, the <em>psaA</em> and <em>tufA</em>
 genes both contain small regions where &gt;15% of residues are edited, 
compared with an average editing rate across each gene of approximately 
4% (<a reveal-id="msu189-F4" data-open="msu189-F4" class="link link-reveal link-table xref-fig">fig. 4</a>).
 To test whether these highly edited sites correspond to particularly 
divergent sequences, we calculated editing frequencies using a sliding 
60-bp window, in polyuridylylated transcripts covering the entire <em>psaA</em> and <em>tufA</em>
 gene sequences. We additionally calculated the predicted sequence 
conservation, over the same sliding window, between the predicted <em>Karl. veneficum psaA</em> and <em>tufA</em> transcript translation products, and the corresponding <em>E. huxleyi</em> protein sequences (<a reveal-id="msu189-F4" data-open="msu189-F4" class="link link-reveal link-table xref-fig">fig. 4</a>). In both genes, editing was specifically correlated with low sequence conservation with <em>E. huxleyi</em> (Pearson correlation = −0.56 for <em>psaA,</em> −0.67 for <em>tufA</em>; <em>P</em> &lt; E-07 for both genes). Notably, over a third of the editing events within <em>tufA</em>
 occur within an 84-bp region, which forms less than one-twelfth of the 
entire gene and is significantly more highly edited than the rest of the
 sequence (chi-squared: <em>P</em> &lt; 0.05). This region corresponds to an in-frame insertion unique to <em>Karl. veneficum</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S5</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). Overall, our data indicate that editing events are associated 
with regions of sequence that are recently acquired or are highly 
divergent. Editing might reduce the effects of these divergent sequences
 on protein function. </p>                        <a id="74404092" scrollto-destination="74404092"></a>
<div id="msu189-F4" data-id="msu189-F4" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 4.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f4p.jpg" alt="Editing is preferentially associated with highly divergent regions of Karlodinium veneficum plastid genes. These graphs compare the frequency of editing with sequence conservation in a 60-bp sliding window over the entire lengths of the Karl. veneficum psaA and tufA genes. The horizontal axis shows the starting position of each window within each gene sequence. The left hand vertical axis of each graph (black line) depicts the total percentage of nucleotide positions within each window that are edited within the transcript sequence. The right hand vertical axis (gray line) shows the proportion of amino acid positions within the predicted translation product of the transcript sequence of this window that are conserved with the predicted translation of the orthologous gene in the Emiliania huxleyi plastid. A table to the right-hand side of each graph shows the total proportion of editing sites over the entire gene, and the Pearson coefficient, and associated significance value of the correlation between sequence conservation and editing frequency. For the tufA gene, correlation coefficients are given both for the complete gene sequence (open figures) and for the gene sequence excluding the highly edited 84-bp insertion region specific to Karl. veneficum (bracketed figures). In all cases, a significant negative correlation is observed." path-from-xml="msu189f4p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404092/msu189f4p.jpeg" path-from-xml="msu189f4p.jpeg" target="_blank">View large</a><a section="74404092" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f4p.jpeg?Expires=1513001585&amp;Signature=DmPOp9ApPLkpeJV1YwohY9fT5FCuIijhQq6QSe180xu7xg6lvbMtkVFPYwdjQQIGFP%7Ez80E34Ep6opzSlEazSzDlIxGBa1FunYFddKNdsaMqpKbPbul8ie4n3d6FD7CiYJP1exUHyE28lF8%7EVPlE0Hi8z%7EdSDkcXIIc7hwqGDGNpagSlgUSKNqYzzRt9OOTRuPxKvsz4EUh5BSpFbkK3A-EiYC0ZAFJGut-i36g4y7qOQNwWex0lGv75PgwnMeURgraq0fv8dAmjJ3UqLhpJ1TLB0FE3JmwRFmiPwN-ZPxHrD5I2lrt9ZfHS9grnO3dFyFLquZY%7EGPSBjk3ieVii8A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404092&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f4p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Editing is preferentially associated with highly divergent regions of <em>Karlodinium veneficum</em>
 plastid genes. These graphs compare the frequency of editing with 
sequence conservation in a 60-bp sliding window over the entire lengths 
of the <em>Karl. veneficum psaA</em> and <em>tufA</em> genes. The 
horizontal axis shows the starting position of each window within each 
gene sequence. The left hand vertical axis of each graph (black line) 
depicts the total percentage of nucleotide positions within each window 
that are edited within the transcript sequence. The right hand vertical 
axis (gray line) shows the proportion of amino acid positions within the
 predicted translation product of the transcript sequence of this window
 that are conserved with the predicted translation of the orthologous 
gene in the <em>Emiliania huxleyi</em> plastid. A table to the 
right-hand side of each graph shows the total proportion of editing 
sites over the entire gene, and the Pearson coefficient, and associated 
significance value of the correlation between sequence conservation and 
editing frequency. For the <em>tufA</em> gene, correlation coefficients 
are given both for the complete gene sequence (open figures) and for the
 gene sequence excluding the highly edited 84-bp insertion region 
specific to <em>Karl. veneficum</em> (bracketed figures). In all cases, a significant negative correlation is observed.</p></div></div><div content-id="msu189-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">F<span class="small-caps">ig</span>. 4.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f4p.jpg" alt="Editing is preferentially associated with highly divergent regions of Karlodinium veneficum plastid genes. These graphs compare the frequency of editing with sequence conservation in a 60-bp sliding window over the entire lengths of the Karl. veneficum psaA and tufA genes. The horizontal axis shows the starting position of each window within each gene sequence. The left hand vertical axis of each graph (black line) depicts the total percentage of nucleotide positions within each window that are edited within the transcript sequence. The right hand vertical axis (gray line) shows the proportion of amino acid positions within the predicted translation product of the transcript sequence of this window that are conserved with the predicted translation of the orthologous gene in the Emiliania huxleyi plastid. A table to the right-hand side of each graph shows the total proportion of editing sites over the entire gene, and the Pearson coefficient, and associated significance value of the correlation between sequence conservation and editing frequency. For the tufA gene, correlation coefficients are given both for the complete gene sequence (open figures) and for the gene sequence excluding the highly edited 84-bp insertion region specific to Karl. veneficum (bracketed figures). In all cases, a significant negative correlation is observed." path-from-xml="msu189f4p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404092/msu189f4p.jpeg" path-from-xml="msu189f4p.jpeg" target="_blank">View large</a><a section="74404092" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f4p.jpeg?Expires=1513001585&amp;Signature=DmPOp9ApPLkpeJV1YwohY9fT5FCuIijhQq6QSe180xu7xg6lvbMtkVFPYwdjQQIGFP%7Ez80E34Ep6opzSlEazSzDlIxGBa1FunYFddKNdsaMqpKbPbul8ie4n3d6FD7CiYJP1exUHyE28lF8%7EVPlE0Hi8z%7EdSDkcXIIc7hwqGDGNpagSlgUSKNqYzzRt9OOTRuPxKvsz4EUh5BSpFbkK3A-EiYC0ZAFJGut-i36g4y7qOQNwWex0lGv75PgwnMeURgraq0fv8dAmjJ3UqLhpJ1TLB0FE3JmwRFmiPwN-ZPxHrD5I2lrt9ZfHS9grnO3dFyFLquZY%7EGPSBjk3ieVii8A__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404092&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f4p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Editing is preferentially associated with highly divergent regions of <em>Karlodinium veneficum</em>
 plastid genes. These graphs compare the frequency of editing with 
sequence conservation in a 60-bp sliding window over the entire lengths 
of the <em>Karl. veneficum psaA</em> and <em>tufA</em> genes. The 
horizontal axis shows the starting position of each window within each 
gene sequence. The left hand vertical axis of each graph (black line) 
depicts the total percentage of nucleotide positions within each window 
that are edited within the transcript sequence. The right hand vertical 
axis (gray line) shows the proportion of amino acid positions within the
 predicted translation product of the transcript sequence of this window
 that are conserved with the predicted translation of the orthologous 
gene in the <em>Emiliania huxleyi</em> plastid. A table to the 
right-hand side of each graph shows the total proportion of editing 
sites over the entire gene, and the Pearson coefficient, and associated 
significance value of the correlation between sequence conservation and 
editing frequency. For the <em>tufA</em> gene, correlation coefficients 
are given both for the complete gene sequence (open figures) and for the
 gene sequence excluding the highly edited 84-bp insertion region 
specific to <em>Karl. veneficum</em> (bracketed figures). In all cases, a significant negative correlation is observed.</p></div></div>                            <h3 scrollto-destination="74404093" id="74404093" class="section-title">Editing-Facilitated Divergent C-Terminal Evolution of <em>Karl. veneficum</em> AtpA</h3>
<p>For the <em>Karl. veneficum atpA</em> gene, editing appears to be involved in the generation of a novel 3′-extension on the conventional CDS (<a reveal-id="msu189-F5" data-open="msu189-F5" class="link link-reveal link-table xref-fig">fig. 5</a>). The <em>Karl. veneficum atpA</em>
 gene contains a premature in-frame TGA codon, which is edited to form a
 CAA-glutamine codon in the mature transcript sequence. However, the <em>Karl. veneficum atpA</em> gene does not contain the consensus 3′-end found in other plastid sequences. The translation product of the <em>Karl. veneficum atpA</em> transcript is similar in sequence up to the final six amino acids in the <em>E. huxleyi</em>
 plastid AtpA protein, where it diverges to contain a 95-aa C-terminal 
extension that bears no homology to any other known sequence (<a reveal-id="msu189-F5" data-open="msu189-F5" class="link link-reveal link-table xref-fig">fig. 5</a>).
 The expression of this extension would be possible only from edited 
transcript sequence, and therefore transcript editing may have enabled 
divergent evolution of the ATP synthase complex in the <em>Karl. veneficum</em> plastid. </p>                        <a id="74404095" scrollto-destination="74404095"></a>
<div id="msu189-F5" data-id="msu189-F5" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 5.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f5p.jpg" alt=" Generation of a novel C-terminal sequence extension by editing of Karlodinium veneficum atpA transcripts. (A) An alignment of the predicted translation products of the genomic and transcript sequences of Karl. veneficum atpA with protein sequences from other plastid lineages. (B) A nucleotide sequence alignment, and predicted translation products of two regions of the Karl. veneficum genomic and transcript sequence in detail. Residues important for defining the size of the predicted translation product of each Karl. veneficum sequence are labeled with vertical arrows. The Karl. veneficum genomic translation product terminates approximately 33-aa upstream of the consensus AtpA C-terminus, due to the presence of an in-frame TGA STOP codon within the atpA gene sequence. This is altered by editing to a CAA-Gln codon (B[i]) in the transcript sequence, enabling the translation of the complete AtpA C-terminus. However, the atpA transcript sequence is highly divergent at the 3′-end, and does not possess a termination codon at the consensus position relative to orthologous AtpA sequences. Instead, it encodes an 85-aa extension sequence that is not conserved with other AtpA sequences, which terminates in an unedited TAA STOP codon (B[ii])." path-from-xml="msu189f5p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404095/msu189f5p.jpeg" path-from-xml="msu189f5p.jpeg" target="_blank">View large</a><a section="74404095" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f5p.jpeg?Expires=1513001585&amp;Signature=LI3O72MJI7Kk4S7I7kHq5GACN3spJIaSt09fZal3SGLincj7rhCGVETOt%7E%7ETycIwvLsyJgnUyWeFTyU-RR%7E6PcBsVFlT81x4y5ybRMzcmVDP7NS-ANU1ngebDbtuRCNWYGQq5%7EZADKrpSHDieLnPNaqisaa3508Pqu8ejmKvyBfK%7EbTdaVRAkh0RyCQzSJIYl%7E43wcI1YbxQrwpsnJMqU%7E74mlgbLgJlOrFCFSR6c8QZt78wERleQzte0rNdat9dj1npGIQmQAZ86dn46HbRpvRsIfs%7EaTw-kUSzRjLq2twrxs%7EC2ONgAMbeFn2iWTeP%7EV9UrEFVVSGfCG47BUzjzg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404095&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f5p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p> Generation of a novel C-terminal sequence extension by editing of <em>Karlodinium veneficum atpA</em> transcripts. (<em>A</em>) An alignment of the predicted translation products of the genomic and transcript sequences of <em>Karl. veneficum atpA</em> with protein sequences from other plastid lineages. (<em>B</em>) A nucleotide sequence alignment, and predicted translation products of two regions of the <em>Karl. veneficum</em>
 genomic and transcript sequence in detail. Residues important for 
defining the size of the predicted translation product of each <em>Karl. veneficum</em> sequence are labeled with vertical arrows. The <em>Karl. veneficum</em>
 genomic translation product terminates approximately 33-aa upstream of 
the consensus AtpA C-terminus, due to the presence of an in-frame TGA 
STOP codon within the <em>atpA</em> gene sequence. This is altered by editing to a CAA-Gln codon (<em>B</em>[<em>i</em>]) in the transcript sequence, enabling the translation of the complete AtpA C-terminus. However, the <em>atpA</em>
 transcript sequence is highly divergent at the 3′-end, and does not 
possess a termination codon at the consensus position relative to 
orthologous AtpA sequences. Instead, it encodes an 85-aa extension 
sequence that is not conserved with other AtpA sequences, which 
terminates in an unedited TAA STOP codon (<em>B</em>[<em>ii</em>]).</p></div></div><div content-id="msu189-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">F<span class="small-caps">ig</span>. 5.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f5p.jpg" alt=" Generation of a novel C-terminal sequence extension by editing of Karlodinium veneficum atpA transcripts. (A) An alignment of the predicted translation products of the genomic and transcript sequences of Karl. veneficum atpA with protein sequences from other plastid lineages. (B) A nucleotide sequence alignment, and predicted translation products of two regions of the Karl. veneficum genomic and transcript sequence in detail. Residues important for defining the size of the predicted translation product of each Karl. veneficum sequence are labeled with vertical arrows. The Karl. veneficum genomic translation product terminates approximately 33-aa upstream of the consensus AtpA C-terminus, due to the presence of an in-frame TGA STOP codon within the atpA gene sequence. This is altered by editing to a CAA-Gln codon (B[i]) in the transcript sequence, enabling the translation of the complete AtpA C-terminus. However, the atpA transcript sequence is highly divergent at the 3′-end, and does not possess a termination codon at the consensus position relative to orthologous AtpA sequences. Instead, it encodes an 85-aa extension sequence that is not conserved with other AtpA sequences, which terminates in an unedited TAA STOP codon (B[ii])." path-from-xml="msu189f5p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404095/msu189f5p.jpeg" path-from-xml="msu189f5p.jpeg" target="_blank">View large</a><a section="74404095" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f5p.jpeg?Expires=1513001585&amp;Signature=LI3O72MJI7Kk4S7I7kHq5GACN3spJIaSt09fZal3SGLincj7rhCGVETOt%7E%7ETycIwvLsyJgnUyWeFTyU-RR%7E6PcBsVFlT81x4y5ybRMzcmVDP7NS-ANU1ngebDbtuRCNWYGQq5%7EZADKrpSHDieLnPNaqisaa3508Pqu8ejmKvyBfK%7EbTdaVRAkh0RyCQzSJIYl%7E43wcI1YbxQrwpsnJMqU%7E74mlgbLgJlOrFCFSR6c8QZt78wERleQzte0rNdat9dj1npGIQmQAZ86dn46HbRpvRsIfs%7EaTw-kUSzRjLq2twrxs%7EC2ONgAMbeFn2iWTeP%7EV9UrEFVVSGfCG47BUzjzg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404095&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f5p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p> Generation of a novel C-terminal sequence extension by editing of <em>Karlodinium veneficum atpA</em> transcripts. (<em>A</em>) An alignment of the predicted translation products of the genomic and transcript sequences of <em>Karl. veneficum atpA</em> with protein sequences from other plastid lineages. (<em>B</em>) A nucleotide sequence alignment, and predicted translation products of two regions of the <em>Karl. veneficum</em>
 genomic and transcript sequence in detail. Residues important for 
defining the size of the predicted translation product of each <em>Karl. veneficum</em> sequence are labeled with vertical arrows. The <em>Karl. veneficum</em>
 genomic translation product terminates approximately 33-aa upstream of 
the consensus AtpA C-terminus, due to the presence of an in-frame TGA 
STOP codon within the <em>atpA</em> gene sequence. This is altered by editing to a CAA-Gln codon (<em>B</em>[<em>i</em>]) in the transcript sequence, enabling the translation of the complete AtpA C-terminus. However, the <em>atpA</em>
 transcript sequence is highly divergent at the 3′-end, and does not 
possess a termination codon at the consensus position relative to 
orthologous AtpA sequences. Instead, it encodes an 85-aa extension 
sequence that is not conserved with other AtpA sequences, which 
terminates in an unedited TAA STOP codon (<em>B</em>[<em>ii</em>]).</p></div></div>                            <h3 scrollto-destination="74404096" id="74404096" class="section-title">Expression and Transcript Processing of Minicircles Located in the <em>Karl. veneficum</em> Plastid</h3>
<p>Certain genes within the <em>Karl. veneficum</em> plastid genome, such as <em>rbcL</em> and <em>dnaK,</em> are enriched in sequencing libraries relative to others (<a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>). These genes have been shown not only to be encoded on the chromosomal <em>Karl. veneficum</em>
 plastid genome sequence but also on multiple small elements, containing
 fragments of individual genes, that do not assemble onto the plastid 
genome (<a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>).The
 episomal elements have been suggested to correspond to a population of 
plastid-located minicircles, which have arisen independently of those 
found in peridinin dinoflagellates (<a reveal-id="msu189-B38" data-open="msu189-B38" class="link link-ref link-reveal xref-bibr">Zhang et al. 1999</a>; <a reveal-id="msu189-B16" data-open="msu189-B16" class="link link-ref link-reveal xref-bibr">Howe et al. 2008</a>; <a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>). However, it is not known whether these episomal elements are located in the <em>Karl. veneficum</em> plastid, nor has a complete episomal element yet been sequenced and confirmed to form a minicircle.</p><p>We investigated whether episomal fragments in <em>Karl. veneficum</em>
 may give rise to polyuridylylated transcripts. Polyuridylylation is not
 found in dinoflagellate nuclei or mitochondria and would accordingly 
confirm localization of the elements to the <em>Karl. veneficum</em> plastid (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. We initially designed primers specific to the chromosomal and episomal copies of <em>rbcL</em>
 and tested for the presence of polyuridylylated transcripts by 
oligo-d(A) primed RT-PCR, as before, but could not identify any evidence
 for poly(U) addition or editing on transcripts of the episomal <em>rbcL</em> elements<em>,</em> in contrast to transcripts of the chromosomal <em>rbcL</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S6</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online).</p><p>We additionally investigated the transcription of episomal <em>dnaK</em> genes. Although there is a complete copy of the <em>rbcL</em> gene within the <em>Karl. veneficum</em> plastid genome, the chromosomal <em>dnaK</em>
 genes lack consensus terminal regions and contain frame-shift 
mutations, suggesting that they do not give rise to translationally 
functional <em>dnaK</em> transcripts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S7A</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>; <a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>). We could not identify polyuridylylated transcripts from either chromosomal <em>dnaK</em> gene. Instead, using PCR primers designed against different regions of <em>dnaK</em> sequence, we identified a single polyuridylylated transcript, which we term <em>dnaK-1</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. 7A</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). The <em>dnaK-1</em> transcript encodes a complete plastid Hsp70 and does not contain any frame-shifts or align with either chromosomal <em>dnaK</em> gene, suggesting that it is expressed from an episomal element.</p><p>To identify what genetic elements might give rise to the <em>dnaK-1</em> transcript, we performed thermal asymmetric interlaced PCR (<a reveal-id="msu189-B25" data-open="msu189-B25" class="link link-ref link-reveal xref-bibr">Liu et al. 1995</a>), using combinations of primers derived from the <em>dnaK-1</em> transcript sequence. We identified a single gene that covered the entire <em>dnaK-1</em> CDS and 3′-UTR past the poly(U) site. The <em>dnaK-1</em> poly(U) site coincides with a genomic T12 motif; however, we identified <em>dnaK-1</em>
 transcripts through circular RT-PCR with poly(U) tails of up to 19 nt 
length, implying that they are generated through posttranscriptional 
sequence modification (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S7B</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). In addition, we found evidence of extensive editing in the <em>dnaK-1</em> transcript sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). Overall, our data imply that <em>dnaK-1</em> is transcribed from a single contiguous genetic element, located within the <em>Karl. veneficum</em> plastid, but separate from the chromosomal genome sequence.</p><p>Surprisingly, the <em>dnaK-1</em> 3′-UTR obtained was found to extend into a region of sequence identical to the 5′-end of the <em>dnaK-1</em> gene, consistent with the <em>dnaK-1</em> gene being located on a plastid minicircle (<a reveal-id="msu189-F6" data-open="msu189-F6" class="link link-reveal link-table xref-fig">fig. 6</a>). The <em>dnaK-1</em> minicircle is 2,323 bp long and contains a single <em>Eco</em>RI restriction site, which is consistent with a 2.3-kbp band containing the <em>dnaK</em> gene identified through Southern blotting of <em>Eco</em>RI-digested <em>Karl. veneficum</em> gDNA (<a reveal-id="msu189-F6" data-open="msu189-F6" class="link link-reveal link-table xref-fig">fig. 6</a>) (<a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>). In addition to a complete <em>dnaK</em> gene, this minicircle contains a <em>Glu<sup>TTC</sup></em> tRNA gene, and a single “high copy” region that is conserved with other episomal sequences previously identified from <em>Karl. veneficum</em> (<a reveal-id="msu189-F6" data-open="msu189-F6" class="link link-reveal link-table xref-fig">fig. 6</a>) (<a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>).
 This is the first complete plastid minicircle identified in a 
fucoxanthin dinoflagellate, confirming that the fucoxanthin plastid 
genome has undergone a similar fragmentation to that observed in 
peridinin dinoflagellate plastid genomes. Our data furthermore show that
 the poly(U) and editing machinery of fucoxanthin dinoflagellates may 
recognize transcripts of genes encoded on minicircles over genes located
 on the chromosomal plastid genome. </p>                        <a id="74404102" scrollto-destination="74404102"></a>
<div id="msu189-F6" data-id="msu189-F6" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 6.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f6p.jpg" alt="Schematic diagram of the Karlodinium veneficum dnaK-1 minicircle. The 2,323-bp dnaK-1 minicircle contains a complete dnaK-1 positioned directly upstream of the predicted “high copy element,” and a GluTCC tRNA gene in the same transcriptional orientation. A single EcoRI restriction site is present on the reverse strand of the tRNA gene." path-from-xml="msu189f6p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404102/msu189f6p.jpeg" path-from-xml="msu189f6p.jpeg" target="_blank">View large</a><a section="74404102" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f6p.jpeg?Expires=1513001585&amp;Signature=DfLyd2AOkaa8JuAw1bJQHW6FdUpwQIh3Ej1JOsbBUhtJ7oEk0pDEowZVALj0-tGyxYyJliQnHvNC-gO1Uyu6mnqxlkVS7JS-Wj4j3QCDL9Kgq1Ba5MgIYAy-Ha0AyTUUexDsw%7EcwhquZlq7JhV%7EGLIUsUNor1vz7dMvsaNxwacQtzrEj49oTbD2lUkWQdUasjUto8gpLGI0N20mn2t7hRcIzamnyGTlqZMnbFqrTeNA2lS7jaDbabLQgcQRXvPhhGi61Jm7h1Fw0qahmnJ40Xlun0t3CUhWczT%7EjX1XhJ0S897JONys4SmlXNQojZsCWgkrRpwvZ3MGeJVogzEk45Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404102&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f6p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the <em>Karlodinium veneficum dnaK-1</em> minicircle. The 2,323-bp <em>dnaK-1</em> minicircle contains a complete <em>dnaK-1</em> positioned directly upstream of the predicted “high copy element,” and a <em>Glu<sup>TCC</sup></em> tRNA gene in the same transcriptional orientation. A single <em>Eco</em>RI restriction site is present on the reverse strand of the tRNA gene.</p></div></div><div content-id="msu189-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">F<span class="small-caps">ig</span>. 6.</div><div class="graphic-wrap"><img class="content-image" src="m_msu189f6p.jpg" alt="Schematic diagram of the Karlodinium veneficum dnaK-1 minicircle. The 2,323-bp dnaK-1 minicircle contains a complete dnaK-1 positioned directly upstream of the predicted “high copy element,” and a GluTCC tRNA gene in the same transcriptional orientation. A single EcoRI restriction site is present on the reverse strand of the tRNA gene." path-from-xml="msu189f6p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic-oup-com.insb.bib.cnrs.fr/view-large/figure/74404102/msu189f6p.jpeg" path-from-xml="msu189f6p.jpeg" target="_blank">View large</a><a section="74404102" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189f6p.jpeg?Expires=1513001585&amp;Signature=DfLyd2AOkaa8JuAw1bJQHW6FdUpwQIh3Ej1JOsbBUhtJ7oEk0pDEowZVALj0-tGyxYyJliQnHvNC-gO1Uyu6mnqxlkVS7JS-Wj4j3QCDL9Kgq1Ba5MgIYAy-Ha0AyTUUexDsw%7EcwhquZlq7JhV%7EGLIUsUNor1vz7dMvsaNxwacQtzrEj49oTbD2lUkWQdUasjUto8gpLGI0N20mn2t7hRcIzamnyGTlqZMnbFqrTeNA2lS7jaDbabLQgcQRXvPhhGi61Jm7h1Fw0qahmnJ40Xlun0t3CUhWczT%7EjX1XhJ0S897JONys4SmlXNQojZsCWgkrRpwvZ3MGeJVogzEk45Q__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q&amp;sec=74404102&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="msu189f6p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic diagram of the <em>Karlodinium veneficum dnaK-1</em> minicircle. The 2,323-bp <em>dnaK-1</em> minicircle contains a complete <em>dnaK-1</em> positioned directly upstream of the predicted “high copy element,” and a <em>Glu<sup>TCC</sup></em> tRNA gene in the same transcriptional orientation. A single <em>Eco</em>RI restriction site is present on the reverse strand of the tRNA gene.</p></div></div>                            <h3 scrollto-destination="74404103" id="74404103" class="section-title">Absence of Poly(U) Addition and Editing from Diatom and Green Algal-Derived Serially Acquired Dinoflagellate Plastids</h3>
<p>We wished to determine whether poly(U) addition and transcript 
editing are found in either dinotom or green dinoflagellate plastids, as
 in the fucoxanthin and peridinin-containing lineages. We performed 
oligo-d(A) primed RT-PCRs on five genes (<em>psbA, psbC, psbD, psaA,</em> and <em>rbcL</em>) using total cellular RNA, and PCR primers specific to the dinotom <em>Kryptoperidinium foliaceum</em> and green dinoflagellate <em>Lepidodinium chlorophorum</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S8A</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). We could not detect polyuridylylated transcripts for any of the genes tested (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S8A</a></span>, lanes 1–5, 7–11, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online). We detected nonpolyuridylylated <em>psbA</em> transcripts in both species using gene-specific cDNA synthesis primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S8A</a></span>, lanes 6 and 12, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online) and by circular RT-PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary fig. S8B</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). We could not find any evidence of editing on these transcript 
sequences. We conclude that poly(U) addition and editing are found only 
in dinoflagellates that possess the ancestral peridinin plastid or the 
fucoxanthin replacement lineage.</p>                            <h2 scrollto-destination="74404105" id="74404105" class="section-title">Discussion</h2>
<p>We have characterized the distribution and function of transcript 
editing and poly(U) tail addition across the entire plastid genome of 
the fucoxanthin dinoflagellate <em>Karl. veneficum</em>. This represents
 the first genome-wide study of transcript processing in a plastid 
acquired through serial endosymbiosis. The demonstration of poly(U) 
addition in <em>Karlodinium</em>, as in <em>Kare. mikimotoi,</em> indicates that it was acquired by a common ancestor of all studied fucoxanthin dinoflagellates (<a reveal-id="msu189-B6" data-open="msu189-B6" class="link link-ref link-reveal xref-bibr">Bergholtz et al. 2006</a>; <a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>).
 We also found extensive sequence editing events, consistent with 
previous studies that identified them in both fucoxanthin dinoflagellate
 species (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012</a>; <a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>). These editing events include transversion substitutions that have not previously been detected in <em>Karl. veneficum</em> but do occur in <em>Kare. mikimotoi</em>, suggesting that these are conserved across extant fucoxanthin dinoflagellates (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012</a>; <a reveal-id="msu189-B19" data-open="msu189-B19" class="link link-ref link-reveal xref-bibr">Jackson et al. 2013</a>).</p><p>Many of the features associated with poly(U) addition and editing in <em>Karl. veneficum</em>
 have previously been documented in peridinin dinoflagellates. Multiple 
types of editing events have already been observed in peridinin 
dinoflagellates, and all species studied have had overall rates of 
editing of under 5%. In all species, A-G and U-C editing have been the 
two most abundant types of editing event (<a reveal-id="msu189-B37" data-open="msu189-B37" class="link link-ref link-reveal xref-bibr">Zauner et al. 2004</a>; <a reveal-id="msu189-B36" data-open="msu189-B36" class="link link-ref link-reveal xref-bibr">Wang and Morse 2006</a>; <a reveal-id="msu189-B40" data-open="msu189-B40" class="link link-ref link-reveal xref-bibr">Dang and Green 2009</a>; <a reveal-id="msu189-B17" data-open="msu189-B17" class="link link-ref link-reveal xref-bibr">Iida et al. 2009</a>; <a reveal-id="msu189-B41" data-open="msu189-B41" class="link link-ref link-reveal xref-bibr">Mungpakdee et al. 2014</a>). As in peridinin dinoflagellates, almost every protein-coding gene within the <em>Karl. veneficum</em> plastid can give rise to polyuridylylated transcripts, whereas tRNA genes do not possess poly(U) sites (<a reveal-id="msu189-B36" data-open="msu189-B36" class="link link-ref link-reveal xref-bibr">Wang and Morse 2006</a>; <a reveal-id="msu189-B29" data-open="msu189-B29" class="link link-ref link-reveal xref-bibr">Nelson et al. 2007</a>; <a reveal-id="msu189-B3" data-open="msu189-B3" class="link link-ref link-reveal xref-bibr">Barbrook et al. 2012</a>).
 Similarly, polyuridylylated polycistronic transcripts and poly(U) sites
 that overlap with adjacent gene sequences have previously been 
identified in peridinin dinoflagellates and in chromerids (<a reveal-id="msu189-B3" data-open="msu189-B3" class="link link-ref link-reveal xref-bibr">Barbrook et al. 2012</a>; <a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>).
 This suggests that poly(U) addition has a similar functional role in 
transcript processing in both peridinin and fucoxanthin dinoflagellate 
plastids.</p><p>We also identified properties of poly(U) addition and 
editing that are specific to fucoxanthin dinoflagellate plastids. The 
editing events in <em>Karl. veneficum</em> include C-A, C-G, and U-A 
editing events that have not previously been detected in any peridinin 
dinoflagellate species although C-A editing has also been detected in <em>Kare. mikimotoi</em> (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. Many of the poly(U) sites within the <em>Karl. veneficum</em> plastid are associated with housekeeping genes, which are not retained in the plastid genomes of peridinin dinoflagellates (<a reveal-id="msu189-B2" data-open="msu189-B2" class="link link-ref link-reveal xref-bibr">Bachvaroff et al. 2004</a>; <a reveal-id="msu189-B16" data-open="msu189-B16" class="link link-ref link-reveal xref-bibr">Howe et al. 2008</a>), and are plastid-located but typically do not possess poly(U) sites in chromerids (<a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>).</p><p>Other unusual transcript processing features are associated with particularly divergent sequences in the <em>Karl. veneficum</em> plastid genome. The absence of poly(U) sites associated with pseudogenes has been described in chromerids (<a reveal-id="msu189-B21" data-open="msu189-B21" class="link link-ref link-reveal xref-bibr">Janouskovec et al. 2013</a>; <a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>),
 but neither this nor a difference in the frequency of editing events on
 functional versus pseudogene transcripts has previously been reported 
in peridinin dinoflagellates. In contrast, at least some pseudogene 
transcripts in peridinin dinoflagellates are known to be extensively 
edited (<a reveal-id="msu189-B17" data-open="msu189-B17" class="link link-ref link-reveal xref-bibr">Iida et al. 2009</a>).
 Poly(U) addition and editing might therefore have a role in 
discriminating functional genes from nonfunctional gene fragments 
generated by recent rearrangements in fucoxanthin dinoflagellate plastid
 genomes. Similarly, the association of editing sites with fast-evolving
 sequences, such as the in-frame insertion in <em>tufA</em>, has not 
been described in other dinoflagellates and contrasts with plastid 
editing in plants, which is predominantly associated with slowly 
evolving sites within the genome sequence (<a reveal-id="msu189-B12" data-open="msu189-B12" class="link link-ref link-reveal xref-bibr">Fujii and Small 2011</a>; <a reveal-id="msu189-B15" data-open="msu189-B15" class="link link-ref link-reveal xref-bibr">Hayes et al. 2012</a>).
 These editing events might help neutralize the effects of 
fast-diverging sequences and recently acquired insertions on protein 
function.</p><p>In other cases, our data indicate that editing and 
poly(U) addition may have indirectly facilitated divergent sequence 
evolution in fucoxanthin dinoflagellate plastids. Sequence editing may 
have permitted the establishment of a novel 3′-sequence extension on 
transcripts of the <em>Karl. veneficum atpA</em> gene. To our knowledge,
 the edited extension of a plastid transcript into nonconserved sequence
 has never previously been reported. Most significantly, we have 
identified one plastid gene—<em>dnaK</em>—for which polyuridylylated and
 edited transcripts are derived from an episomal minicircle. This 
represents the first complete plastid minicircle sequence from a 
fucoxanthin dinoflagellate and suggests that the plastid genomes of 
fucoxanthin and peridinin dinoflagellates are undergoing convergent 
evolution events (<a reveal-id="msu189-B38" data-open="msu189-B38" class="link link-ref link-reveal xref-bibr">Zhang et al. 1999</a>; <a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>). The preferential targeting of the poly(U) and editing to <em>dnaK</em>
 gene copies located on minicircles may have led to their fixation over 
copies located on the chromosomal plastid genome, which appear to have 
been reduced to pseudogenes (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>; <a reveal-id="msu189-B11" data-open="msu189-B11" class="link link-ref link-reveal xref-bibr">Espelund et al. 2012</a>).</p><p>Overall, our data indicate that poly(U) addition and editing in <em>Karl. veneficum</em>
 have evolved dynamically alongside the underlying genome, reducing the 
effects of mutations on plastid function, and potentially enabling the 
evolutionary fixation of divergent sequences. It remains to be seen 
whether poly(U) addition and editing were acquired after the extremely 
fast sequence evolution observed in the <em>Karl. veneficum</em> plastid
 commenced, or whether fucoxanthin plastid genomes and transcript 
processing have a more tightly interconnected evolutionary history. This
 might be resolved by investigating genome and transcriptome evolution 
in <em>Kare. mikimotoi</em>, or other less well-characterized fucoxanthin dinoflagellate plastids (<a reveal-id="msu189-B34" data-open="msu189-B34" class="link link-ref link-reveal xref-bibr">Takishita et al. 1999</a>; <a reveal-id="msu189-B6" data-open="msu189-B6" class="link link-ref link-reveal xref-bibr">Bergholtz et al. 2006</a>).
 Notably, the serially acquired plastids of dinotoms, which have less 
divergent genome sequences than fucoxanthin dinoflagellates, and of <em>Lepidodinium</em>, do not apply poly(U) tails or edit plastid transcripts. It will be worth determining whether the dinotoms, or <em>Lepidodinium,</em>
 have retained any factors involved in plastid gene expression from the 
ancestral peridinin symbiosis, for example by reinspecting existing 
transcriptome data (<a reveal-id="msu189-B27" data-open="msu189-B27" class="link link-ref link-reveal xref-bibr">Minge et al. 2010</a>; <a reveal-id="msu189-B7" data-open="msu189-B7" class="link link-ref link-reveal xref-bibr">Burki et al. 2014</a>).
 Further studies of dinoflagellates that have undergone serial 
endosymbiosis may provide important insights into the coevolution of 
plastid genomes and gene expression pathways.</p>                            <h2 scrollto-destination="74404112" id="74404112" class="section-title">Materials and Methods</h2>
                            <h3 scrollto-destination="74404113" id="74404113" class="section-title">Cultures</h3>
<p><em>Karlodinium veneficum</em> RCC2539 (also listed as UIO297) and <em>L. chlorophorum</em> (AC195) were grown in modified k/2 medium, as previously described (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>, under 50 μE m<sup>−2 </sup>s<sup>−1</sup> continuous light at a controlled temperature of 15 °C. <em>Kryptoperidinium</em> (<em>Glenodinium</em>) <em>foliaceum</em> PCC499 was grown in f/2 medium, under a 30 μE m<sup>−2 </sup>s<sup>−1</sup> 12:12 light:dark cycle, at 15–20 °C. To confirm the identity of the <em>Karl. veneficum</em> culture, molecular barcode sequences were generated by PCR of genomic DNA for multiple loci in the <em>Karl. veneficum</em> plastid genome. These were found to be identical to the previously published <em>Karl. veneficum</em> plastid genome sequence (strain UIO083).</p>                            <h3 scrollto-destination="74404115" id="74404115" class="section-title">Nucleic Acid Isolation</h3>
<p>Nucleic acids were isolated from cultures of each species harvested 
in early stationary phase (ca. 30–60 days postinoculation). Cells were 
pelleted by centrifugation and washed in sterile growth medium. For RNA 
isolation, 50 mg pellets of each culture were resuspended in 1 ml TRIzol
 reagent (Ambion), and frozen at −80 °C and thawed on ice to lyse the 
cells. Total cellular RNA was then isolated by phase extraction, DNase 
treated and cleaned with an RNeasy column (Qiagen) as previously 
described (<a reveal-id="msu189-B3" data-open="msu189-B3" class="link link-ref link-reveal xref-bibr">Barbrook et al. 2012</a>; <a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>). Genomic DNA was isolated from cell pellets by phase extraction and cleaned with a DNeasy column as previously described (<a reveal-id="msu189-B4" data-open="msu189-B4" class="link link-ref link-reveal xref-bibr">Barbrook and Howe 2000</a>; <a reveal-id="msu189-B28" data-open="msu189-B28" class="link link-ref link-reveal xref-bibr">Nash et al. 2007</a>).</p><p>The
 concentration of each nucleic acid obtained was quantified using a 
nanodrop spectrophotometer. RNA integrity was confirmed by 
electrophoresis of 1 μg of each sample in an RNase-free 1% agarose gel 
containing 0.003% volumes of ethidium bromide. To determine whether any 
sample contained residual DNA contamination, each RNA sample was used as
 the direct substrate for a PCR using internal primers against the <em>psbA</em>
 gene of each sequence. Only samples for which negative results were 
observed in the initial PCR, and in the product of a reamplification PCR
 using the initial product as a PCR template, were used for further 
experimentation.</p>                            <h3 scrollto-destination="74404118" id="74404118" class="section-title">RT-PCR and Sequencing</h3>
<p>Reverse transcription was performed using Superscript III (Life Technologies), as previously described (<a reveal-id="msu189-B9" data-open="msu189-B9" class="link link-ref link-reveal xref-bibr">Dorrell et al. 2014</a>).
 cDNA was synthesized either with an oligo-d(A) primer, to generate 
products from polyuridylylated transcripts as previously described (<a reveal-id="msu189-B3" data-open="msu189-B3" class="link link-ref link-reveal xref-bibr">Barbrook et al. 2012</a>)
 or with internal primers specific to a particular plastid gene. PCR was
 performed with GoTaq flexi polymerase (Promega) as previously described
 (<a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>. PCR primers used are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online. Circular RT-PCR of <em>Karl. veneficum</em> transcripts and thermal asymmetric interlaced PCR of <em>dnaK</em> genetic elements were performed as previously described (<a reveal-id="msu189-B25" data-open="msu189-B25" class="link link-ref link-reveal xref-bibr">Liu et al. 1995</a>; <a reveal-id="msu189-B10" data-open="msu189-B10" class="link link-ref link-reveal xref-bibr">Dorrell and Howe 2012)</a>.</p><p>PCR
 products were visualized by electrophoresis in a 1% agarose-TBE gel 
containing ethidium bromide. Products were directly purified using a 
QIAquick column elution kit (Qiagen). Where multiple bands were 
detectable, individual products were separated by electrophoresis, cut 
out of the agarose gel, and purified as before. Products were sequenced 
using an Applied Biosystems 3730xl DNA Analyzer. The sequences of three 
polyuridylylated transcript sequences (<em>psaC, psbI</em>, and <em>psbK</em>) and one internal transcript sequence (<em>ORF4</em>) that were too short to be uploaded to GenBank are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span> online.</p>                            <h3 scrollto-destination="74404121" id="74404121" class="section-title">Sequence Analysis</h3>
<p>Potential recombination events associated with the <em>Karl. veneficum</em>
 plastid were identified by comparison of the complete plastid genome 
sequence with the complete plastid genomes of the free-living 
haptophytes <em>E. huxleyi, P. globosa, Pavlova lutheri</em>, and the partial plastid genome of the uncultured prymnesiophyte C19487 (<a reveal-id="msu189-B30" data-open="msu189-B30" class="link link-ref link-reveal xref-bibr">Puerta et al. 2005</a>; <a reveal-id="msu189-B5" data-open="msu189-B5" class="link link-ref link-reveal xref-bibr">Baurain et al. 2010</a>; <a reveal-id="msu189-B8" data-open="msu189-B8" class="link link-ref link-reveal xref-bibr">Cuvelier et al. 2010</a>).</p><p>Poly(U) sites were identified by aligning the sequences of the oligo-d(A) RT-PCR products against the published <em>Karl. veneficum</em> plastid genome sequence (<a reveal-id="msu189-B13" data-open="msu189-B13" class="link link-ref link-reveal xref-bibr">Gabrielsen et al. 2011</a>) using GENEious (<a class="link link-uri" href="http://www.geneious.com/" target="">www.geneious.com</a>).
 To identify motifs that might be associated with poly(U) sites, 
alignments were constructed of the 3′-UTR of each polyuridylylated 
transcript and of the first 100 bp downstream of the poly(U) site (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). As a negative control, sequence alignments were constructed 
using the first 100 bp of the 3′-UTR sequence of each gene found not to 
have a poly(U) site (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">supplementary table S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q">Supplementary Material</a></span>
 online). The presence of primary sequence motifs that might be 
associated with poly(U) sites was investigated by reciprocal BLASTn 
searches in each alignment, and conserved RNA secondary structures were 
searched for using the WAR server (<a class="link link-uri" href="http://genome.ku.dk/resources/war/" target="">http://genome.ku.dk/resources/war/</a>, last accessed June 18, 2014) (<a reveal-id="msu189-B35" data-open="msu189-B35" class="link link-ref link-reveal xref-bibr">Torarinsson and Lindgreen 2008</a>).
 The relative GC and purine/pyrimidine contents of each sequence were 
quantified using GENEious, and the minimum Gibbs free energy of folding 
of each sequence was calculated using the mFold server (<a class="link link-uri" href="http://mfold.rna.albany.edu/" target="">http://mfold.rna.albany.edu</a>, last accessed June 18, 2014) (<a reveal-id="msu189-B39" data-open="msu189-B39" class="link link-ref link-reveal xref-bibr">Zuker 2003</a>).</p>                            <h3 scrollto-destination="74404124" id="74404124" class="section-title">Editing Analysis</h3>
<p>Sequence editing was quantified for each gene by GENEious alignments 
of transcript and genomic sequences. The predicted effect of editing on 
protein sequence was determined by in silico translation. To determine 
the effect of transcript editing on protein sequence conservation 
between <em>Karl. veneficum</em> and haptophyte orthologs, conceptual translation sequences of the transcript and genomic sequence of each gene in the <em>Karl. veneficum</em> were aligned to plastid protein sequences from the haptophytes <em>E. huxleyi</em> and <em>P. globosa</em> using BLAST (<a reveal-id="msu189-B30" data-open="msu189-B30" class="link link-ref link-reveal xref-bibr">Puerta et al. 2005</a>). For each alignment, the number of residues conserved between the <em>Karl. veneficum</em>
 and haptophyte protein sequences were recorded. Identical amino acids 
between the two species at any position were scored as a complete match,
 and positives were scored as a 50% match.</p><p>To determine whether editing sites were clustered within certain regions of <em>Karl. veneficum</em> plastid genes, transcript sequences covering the entire CDS of the <em>psaA</em> and <em>tufA</em>
 genes were obtained by RT-PCR and aligned to the corresponding genomic 
sequences. Editing sites were identified in each alignment, and scored 
over a 60-bp sliding sequence window, and regions with elevated 
frequencies of editing relative to the entire CDS were identified by a 
binomial test. Sequence conservation between the <em>Karl. veneficum</em> and <em>E. huxleyi</em>
 protein sequences was scored over each window using BLAST alignment, as
 before. The total number of matching positions were summed over each 
60-bp sliding window, and the Pearson correlation coefficients between 
the degree of sequence conservation and proportion of edited residues 
over each gene were calculated.</p>                            <h2 scrollto-destination="74404127" id="74404127" class="backacknowledgements-title">Acknowledgments</h2>
<p>This work was supported by ASSEMBLE (EFETPIDC), and the BBSRC 
(BB/F017464/1 to R.G.D.). The authors thank two anonymous referees for 
their helpful comments on the manuscript. This work was supported by 
ASSEMBLE (EFETPIDC), the Biotechnology and Biological Sciences Research 
Council (BB/F017464/1 to R.G.D.) and UCAM OpenAccess (OA-1045).</p>                            <h2 scrollto-destination="74404129" id="74404129" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="msu189-B1" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B1"><div class="ref false"><div role="msu189-B1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asakura</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Bayraktar</div> <div class="given-names">OA</div></div>,&nbsp; <div class="name"><div class="surname">Barkan</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Two CRM protein subfamilies cooperate in the splicing of group IIB introns in chloroplasts</div>,&nbsp;<div class="source">RNA</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">2319</div>-<div class="lpage">2332</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Two%20CRM%20protein%20subfamilies%20cooperate%20in%20the%20splicing%20of%20group%20IIB%20introns%20in%20chloroplasts&amp;author=Y%20Asakura&amp;author=OA%20Bayraktar&amp;author=A%20Barkan&amp;publication_year=2008&amp;journal=RNA&amp;volume=14&amp;pages=2319-2332" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1261/rna.1223708" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.1223708" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1261%2Frna.1223708"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/18799595" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B2" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B2"><div class="ref false"><div role="msu189-B2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bachvaroff</div> <div class="given-names">TR</div></div>,&nbsp; <div class="name"><div class="surname">Concepcion</div> <div class="given-names">GT</div></div>,&nbsp; <div class="name"><div class="surname">Rogers</div> <div class="given-names">CR</div></div>,&nbsp; <div class="name"><div class="surname">Herman</div> <div class="given-names">EM</div></div>,&nbsp; <div class="name"><div class="surname">Delwiche</div> <div class="given-names">CF</div></div>.&nbsp;</div><div class="article-title">Dinoflagellate expressed indicate massive transfer to the nuclear genome sequence tag data of chloroplast genes</div>,&nbsp;<div class="source">Protist</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">155</div>&nbsp;(pg.&nbsp;<div class="fpage">65</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Dinoflagellate%20expressed%20indicate%20massive%20transfer%20to%20the%20nuclear%20genome%20sequence%20tag%20data%20of%20chloroplast%20genes&amp;author=TR%20Bachvaroff&amp;author=GT%20Concepcion&amp;author=CR%20Rogers&amp;author=EM%20Herman&amp;author=CF%20Delwiche&amp;publication_year=2004&amp;journal=Protist&amp;volume=155&amp;pages=65-78" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1078/1434461000165" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1078%2F1434461000165" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1078%2F1434461000165"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/15144059" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B3" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B3"><div class="ref false"><div role="msu189-B3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barbrook</div> <div class="given-names">AC</div></div>,&nbsp; <div class="name"><div class="surname">Dorrell</div> <div class="given-names">RG</div></div>,&nbsp; <div class="name"><div class="surname">Burrows</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Plenderleith</div> <div class="given-names">LJ</div></div>,&nbsp; <div class="name"><div class="surname">Nisbet</div> <div class="given-names">RER</div></div>,&nbsp; <div class="name"><div class="surname">Howe</div> <div class="given-names">CJ</div></div>.&nbsp;</div><div class="article-title">Polyuridylylation and processing of transcripts from multiple gene minicircles in chloroplasts of the dinoflagellate <em>Amphidinium carterae</em></div>,&nbsp;<div class="source">Plant Mol Biol.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">79</div>&nbsp;(pg.&nbsp;<div class="fpage">347</div>-<div class="lpage">357</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Polyuridylylation%20and%20processing%20of%20transcripts%20from%20multiple%20gene%20minicircles%20in%20chloroplasts%20of%20the%20dinoflagellate%20Amphidinium%20carterae&amp;author=AC%20Barbrook&amp;author=RG%20Dorrell&amp;author=J%20Burrows&amp;author=LJ%20Plenderleith&amp;author=RER%20Nisbet&amp;author=CJ%20Howe&amp;publication_year=2012&amp;journal=Plant%20Mol%20Biol.&amp;volume=79&amp;pages=347-357" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1007/s11103-012-9916-z" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-012-9916-z" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs11103-012-9916-z"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/22562591" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B4" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B4"><div class="ref false"><div role="msu189-B4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barbrook</div> <div class="given-names">AC</div></div>,&nbsp; <div class="name"><div class="surname">Howe</div> <div class="given-names">CJ</div></div>.&nbsp;</div><div class="article-title">Minicircular plastid DNA in the dinoflagellate <em>Amphidinium operculatum</em></div>,&nbsp;<div class="source">Mol Gen Genet.</div>&nbsp;,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">263</div>&nbsp;(pg.&nbsp;<div class="fpage">152</div>-<div class="lpage">158</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Minicircular%20plastid%20DNA%20in%20the%20dinoflagellate%20Amphidinium%20operculatum&amp;author=AC%20Barbrook&amp;author=CJ%20Howe&amp;publication_year=2000&amp;journal=Mol%20Gen%20Genet.&amp;volume=263&amp;pages=152-158" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1007/s004380050042" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380050042" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs004380050042"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/10732684" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B5" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B5"><div class="ref false"><div role="msu189-B5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baurain</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Brinkmann</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Petersen</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Rodriguez-Ezpeleta</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Stechmann</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Demoulin</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Roger</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Burger</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Lang</div> <div class="given-names">BF</div></div>,&nbsp; <div class="name"><div class="surname">Philippe</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Phylogenomic evidence for separate acquisition of plastids in cryptophytes, haptophytes, and stramenopiles</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">1698</div>-<div class="lpage">1709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Phylogenomic%20evidence%20for%20separate%20acquisition%20of%20plastids%20in%20cryptophytes%2C%20haptophytes%2C%20and%20stramenopiles&amp;author=D%20Baurain&amp;author=H%20Brinkmann&amp;author=J%20Petersen&amp;author=N%20Rodriguez-Ezpeleta&amp;author=A%20Stechmann&amp;author=V%20Demoulin&amp;author=AJ%20Roger&amp;author=G%20Burger&amp;author=BF%20Lang&amp;author=H%20Philippe&amp;publication_year=2010&amp;journal=Mol%20Biol%20Evol.&amp;volume=27&amp;pages=1698-1709" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/molbev/msq059" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq059" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsq059"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/20194427" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B6" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B6"><div class="ref false"><div role="msu189-B6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bergholtz</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Daugbjerg</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Moestrup</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Fernandez-Tejedor</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">On the identity of <em>Karlodinium veneficum</em> and description of <em>Karlodinium armiger</em> sp nov (Dinophyceae), based on light and electron microscopy, nuclear-encoded LSU rDNA, and pigment composition</div>,&nbsp;<div class="source">J Phycol.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">42</div>&nbsp;(pg.&nbsp;<div class="fpage">170</div>-<div class="lpage">193</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=On%20the%20identity%20of%20Karlodinium%20veneficum%20and%20description%20of%20Karlodinium%20armiger%20sp%20nov%20%28Dinophyceae%29%2C%20based%20on%20light%20and%20electron%20microscopy%2C%20nuclear-encoded%20LSU%20rDNA%2C%20and%20pigment%20composition&amp;author=T%20Bergholtz&amp;author=N%20Daugbjerg&amp;author=O%20Moestrup&amp;author=M%20Fernandez-Tejedor&amp;publication_year=2006&amp;journal=J%20Phycol.&amp;volume=42&amp;pages=170-193" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1111/jpy.2006.42.issue-1" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjpy.2006.42.issue-1" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fjpy.2006.42.issue-1"> </span></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B7" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B7"><div class="ref false"><div role="msu189-B7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burki</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Imanian</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Hehenberger</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Maruyama</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Keeling</div> <div class="given-names">PJ</div></div>.&nbsp;</div><div class="article-title">Endosymbiotic gene transfer in tertiary plastid-containing dinoflagellates</div>,&nbsp;<div class="source">Eukaryot Cell.</div>&nbsp;,&nbsp;<div class="year">2014</div>,&nbsp;vol.&nbsp;<div class="volume">13</div>&nbsp;(pg.&nbsp;<div class="fpage">246</div>-<div class="lpage">255</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Endosymbiotic%20gene%20transfer%20in%20tertiary%20plastid-containing%20dinoflagellates&amp;author=F%20Burki&amp;author=B%20Imanian&amp;author=E%20Hehenberger&amp;author=Y%20Hirakawa&amp;author=S%20Maruyama&amp;author=PJ%20Keeling&amp;publication_year=2014&amp;journal=Eukaryot%20Cell.&amp;volume=13&amp;pages=246-255" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1128/EC.00299-13" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00299-13" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FEC.00299-13"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/24297445" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B8" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B8"><div class="ref false"><div role="msu189-B8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cuvelier</div> <div class="given-names">ML</div></div>,&nbsp; <div class="name"><div class="surname">Allen</div> <div class="given-names">AE</div></div>,&nbsp; <div class="name"><div class="surname">Monier</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">McCrow</div> <div class="given-names">JP</div></div>,&nbsp; <div class="name"><div class="surname">Messie</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Tringe</div> <div class="given-names">SG</div></div>,&nbsp; <div class="name"><div class="surname">Woyke</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Welsh</div> <div class="given-names">RM</div></div>,&nbsp; <div class="name"><div class="surname">Ishoey</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Lee</div> <div class="given-names">JH</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Targeted metagenomics and ecology of globally important uncultured eukaryotic phytoplankton</div>,&nbsp;<div class="source">Proc Natl Acad Sci U S A.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">107</div>&nbsp;(pg.&nbsp;<div class="fpage">14679</div>-<div class="lpage">14684</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Targeted%20metagenomics%20and%20ecology%20of%20globally%20important%20uncultured%20eukaryotic%20phytoplankton&amp;author=ML%20Cuvelier&amp;author=AE%20Allen&amp;author=A%20Monier&amp;author=JP%20McCrow&amp;author=M%20Messie&amp;author=SG%20Tringe&amp;author=T%20Woyke&amp;author=RM%20Welsh&amp;author=T%20Ishoey&amp;author=JH%20Lee&amp;publication_year=2010&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A.&amp;volume=107&amp;pages=14679-14684" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1073/pnas.1001665107" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1001665107" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1001665107"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/20668244" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B40" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B40"><div class="ref false"><div role="msu189-B40" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dang</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Green</div> <div class="given-names">BR</div></div>.&nbsp;</div><div class="article-title">Substitutional editing of <em>Heterocapsa triquetra</em> chloroplast transcripts and a folding model for its divergent chloroplast 16S rRNA</div>,&nbsp;<div class="source">Gene</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">442</div>&nbsp;(pg.&nbsp;<div class="fpage">73</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Substitutional%20editing%20of%20Heterocapsa%20triquetra%20chloroplast%20transcripts%20and%20a%20folding%20model%20for%20its%20divergent%20chloroplast%2016S%20rRNA&amp;author=Y%20Dang&amp;author=BR%20Green&amp;publication_year=2009&amp;journal=Gene&amp;volume=442&amp;pages=73-80" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1016/j.gene.2009.04.006" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2009.04.006" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2009.04.006"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/19376212" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B9" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B9"><div class="ref false"><div role="msu189-B9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dorrell</div> <div class="given-names">RG</div></div>,&nbsp; <div class="name"><div class="surname">Drew</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Nisbet</div> <div class="given-names">RE</div></div>,&nbsp; <div class="name"><div class="surname">Howe</div> <div class="given-names">CJ</div></div>.&nbsp;</div><div class="article-title">Evolution of chloroplast transcript processing in <em>Plasmodium</em> and its chromerid algal relatives</div>,&nbsp;<div class="source">PLoS Genet.</div>&nbsp;,&nbsp;<div class="year">2014</div>,&nbsp;vol.&nbsp;<div class="volume">10</div>&nbsp;pg.&nbsp;<div class="fpage">e1004008</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Evolution%20of%20chloroplast%20transcript%20processing%20in%20Plasmodium%20and%20its%20chromerid%20algal%20relatives&amp;author=RG%20Dorrell&amp;author=J%20Drew&amp;author=RE%20Nisbet&amp;author=CJ%20Howe&amp;publication_year=2014&amp;journal=PLoS%20Genet.&amp;volume=10&amp;pages=e1004008" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1371/journal.pgen.1004008" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004008" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1004008"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/24453981" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B10" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B10"><div class="ref false"><div role="msu189-B10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dorrell</div> <div class="given-names">RG</div></div>,&nbsp; <div class="name"><div class="surname">Howe</div> <div class="given-names">CJ</div></div>.&nbsp;</div><div class="article-title">Functional remodeling of RNA processing in replacement chloroplasts by pathways retained from their predecessors</div>,&nbsp;<div class="source">Proc Natl Acad Sci U S A.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">109</div>&nbsp;(pg.&nbsp;<div class="fpage">18879</div>-<div class="lpage">18884</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Functional%20remodeling%20of%20RNA%20processing%20in%20replacement%20chloroplasts%20by%20pathways%20retained%20from%20their%20predecessors&amp;author=RG%20Dorrell&amp;author=CJ%20Howe&amp;publication_year=2012&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A.&amp;volume=109&amp;pages=18879-18884" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1073/pnas.1212270109" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1212270109" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1212270109"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/23112181" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B11" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B11"><div class="ref false"><div role="msu189-B11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Espelund</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Minge</div> <div class="given-names">MA</div></div>,&nbsp; <div class="name"><div class="surname">Gabrielsen</div> <div class="given-names">TM</div></div>,&nbsp; <div class="name"><div class="surname">Nederbragt</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Shalchian-Tabrizi</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Otis</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Turmel</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Lemieux</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Jakobsen</div> <div class="given-names">KS</div></div>.&nbsp;</div><div class="article-title">Genome fragmentation is not confined to the peridinin plastid in dinoflagellates</div>,&nbsp;<div class="source">PLoS One</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">7</div>&nbsp;pg.&nbsp;<div class="fpage">e38809</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Genome%20fragmentation%20is%20not%20confined%20to%20the%20peridinin%20plastid%20in%20dinoflagellates&amp;author=M%20Espelund&amp;author=MA%20Minge&amp;author=TM%20Gabrielsen&amp;author=AJ%20Nederbragt&amp;author=K%20Shalchian-Tabrizi&amp;author=C%20Otis&amp;author=M%20Turmel&amp;author=C%20Lemieux&amp;author=KS%20Jakobsen&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e38809" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1371/journal.pone.0038809" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0038809" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0038809"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/22719952" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B12" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B12"><div class="ref false"><div role="msu189-B12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujii</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Small</div> <div class="given-names">I</div></div>.&nbsp;</div><div class="article-title">The evolution of RNA editing and pentatricopeptide repeat genes</div>,&nbsp;<div class="source">New Phytol.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">191</div>&nbsp;(pg.&nbsp;<div class="fpage">37</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=The%20evolution%20of%20RNA%20editing%20and%20pentatricopeptide%20repeat%20genes&amp;author=S%20Fujii&amp;author=I%20Small&amp;publication_year=2011&amp;journal=New%20Phytol.&amp;volume=191&amp;pages=37-47" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1111/j.1469-8137.2011.03746.x" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2011.03746.x" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1469-8137.2011.03746.x"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/21557747" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B13" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B13"><div class="ref false"><div role="msu189-B13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gabrielsen</div> <div class="given-names">TM</div></div>,&nbsp; <div class="name"><div class="surname">Minge</div> <div class="given-names">MA</div></div>,&nbsp; <div class="name"><div class="surname">Espelund</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Tooming-Klunderud</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Patil</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Nederbragt</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Otis</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Turmel</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Shalchian-Tabrizi</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Lemieux</div> <div class="given-names">C</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Genome evolution of a tertiary dinoflagellate plastid</div>,&nbsp;<div class="source">PLoS One</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;pg.&nbsp;<div class="fpage">e19132</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Genome%20evolution%20of%20a%20tertiary%20dinoflagellate%20plastid&amp;author=TM%20Gabrielsen&amp;author=MA%20Minge&amp;author=M%20Espelund&amp;author=A%20Tooming-Klunderud&amp;author=V%20Patil&amp;author=AJ%20Nederbragt&amp;author=C%20Otis&amp;author=M%20Turmel&amp;author=K%20Shalchian-Tabrizi&amp;author=C%20Lemieux&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e19132" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1371/journal.pone.0019132" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019132" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0019132"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/21541332" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B14" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B14"><div class="ref false"><div role="msu189-B14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Green</div> <div class="given-names">BR</div></div>.&nbsp;</div><div class="article-title">Chloroplast genomes of photosynthetic eukaryotes</div>,&nbsp;<div class="source">Plant J.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">66</div>&nbsp;(pg.&nbsp;<div class="fpage">34</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Chloroplast%20genomes%20of%20photosynthetic%20eukaryotes&amp;author=BR%20Green&amp;publication_year=2011&amp;journal=Plant%20J.&amp;volume=66&amp;pages=34-44" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1111/tpj.2011.66.issue-1" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2011.66.issue-1" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.2011.66.issue-1"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/21443621" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B15" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B15"><div class="ref false"><div role="msu189-B15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayes</div> <div class="given-names">ML</div></div>,&nbsp; <div class="name"><div class="surname">Giang</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Mulligan</div> <div class="given-names">RM</div></div>.&nbsp;</div><div class="article-title">Molecular
 evolution of pentatricopeptide repeat genes reveals truncation in 
species lacking an editing target and structural domains under distinct 
selective pressures</div>,&nbsp;<div class="source">BMC Evol Biol.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;pg.&nbsp;<div class="fpage">13</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Molecular%20evolution%20of%20pentatricopeptide%20repeat%20genes%20reveals%20truncation%20in%20species%20lacking%20an%20editing%20target%20and%20structural%20domains%20under%20distinct%20selective%20pressures&amp;author=ML%20Hayes&amp;author=K%20Giang&amp;author=RM%20Mulligan&amp;publication_year=2012&amp;journal=BMC%20Evol%20Biol.&amp;volume=12&amp;pages=13" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1186/1471-2148-12-66" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-12-66" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2148-12-66"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/22284070" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B16" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B16"><div class="ref false"><div role="msu189-B16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Howe</div> <div class="given-names">CJ</div></div>,&nbsp; <div class="name"><div class="surname">Nisbet</div> <div class="given-names">RER</div></div>,&nbsp; <div class="name"><div class="surname">Barbrook</div> <div class="given-names">AC</div></div>.&nbsp;</div><div class="article-title">The remarkable chloroplast genome of dinoflagellates</div>,&nbsp;<div class="source">J Exp Bot.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">59</div>&nbsp;(pg.&nbsp;<div class="fpage">1035</div>-<div class="lpage">1045</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=The%20remarkable%20chloroplast%20genome%20of%20dinoflagellates&amp;author=CJ%20Howe&amp;author=RER%20Nisbet&amp;author=AC%20Barbrook&amp;publication_year=2008&amp;journal=J%20Exp%20Bot.&amp;volume=59&amp;pages=1035-1045" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/jxb/erm292" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm292" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Ferm292"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/18319241" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B17" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B17"><div class="ref false"><div role="msu189-B17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Kobiyama</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Ogata</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Murakami</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Identification of transcribed and persistent variants of the <em>psbA</em> gene carried by plastid minicircles in a dinoflagellate</div>,&nbsp;<div class="source">Curr Genet.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">55</div>&nbsp;(pg.&nbsp;<div class="fpage">583</div>-<div class="lpage">591</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Identification%20of%20transcribed%20and%20persistent%20variants%20of%20the%20psbA%20gene%20carried%20by%20plastid%20minicircles%20in%20a%20dinoflagellate&amp;author=S%20Iida&amp;author=A%20Kobiyama&amp;author=T%20Ogata&amp;author=A%20Murakami&amp;publication_year=2009&amp;journal=Curr%20Genet.&amp;volume=55&amp;pages=583-591" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1007/s00294-009-0271-9" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00294-009-0271-9" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00294-009-0271-9"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/19756629" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B18" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B18"><div class="ref false"><div role="msu189-B18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Imanian</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Pombert</div> <div class="given-names">JF</div></div>,&nbsp; <div class="name"><div class="surname">Keeling</div> <div class="given-names">PJ</div></div>.&nbsp;</div><div class="article-title">The complete plastid genomes of the two ‘dinotoms’ <em>Durinskia baltica</em> and <em>Kryptoperidinium foliaceum</em></div>,&nbsp;<div class="source">PLoS One</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">5</div>&nbsp;pg.&nbsp;<div class="fpage">e10711</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=The%20complete%20plastid%20genomes%20of%20the%20two%20%E2%80%98dinotoms%E2%80%99%20Durinskia%20baltica%20and%20Kryptoperidinium%20foliaceum&amp;author=B%20Imanian&amp;author=JF%20Pombert&amp;author=PJ%20Keeling&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e10711" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1371/journal.pone.0010711" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0010711" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0010711"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/20502706" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B19" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B19"><div class="ref false"><div role="msu189-B19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">CJ</div></div>,&nbsp; <div class="name"><div class="surname">Gornik</div> <div class="given-names">SG</div></div>,&nbsp; <div class="name"><div class="surname">Waller</div> <div class="given-names">RF</div></div>.&nbsp;</div><div class="article-title">A tertiary plastid gains RNA editing in its new host</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">30</div>&nbsp;(pg.&nbsp;<div class="fpage">788</div>-<div class="lpage">792</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=A%20tertiary%20plastid%20gains%20RNA%20editing%20in%20its%20new%20host&amp;author=CJ%20Jackson&amp;author=SG%20Gornik&amp;author=RF%20Waller&amp;publication_year=2013&amp;journal=Mol%20Biol%20Evol.&amp;volume=30&amp;pages=788-792" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/molbev/mss270" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss270" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmss270"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/23197592" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B20" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B20"><div class="ref false"><div role="msu189-B20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Janouskovec</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Horák</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Oborník</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Lukes</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Keeling</div> <div class="given-names">PJ</div></div>.&nbsp;</div><div class="article-title">A common red algal origin of the apicomplexan, dinoflagellate, and heterokont plastids</div>,&nbsp;<div class="source">Proc Natl Acad Sci U S A.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">107</div>&nbsp;(pg.&nbsp;<div class="fpage">10949</div>-<div class="lpage">10954</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=A%20common%20red%20algal%20origin%20of%20the%20apicomplexan%2C%20dinoflagellate%2C%20and%20heterokont%20plastids&amp;author=J%20Janouskovec&amp;author=A%20Hor%C3%A1k&amp;author=M%20Oborn%C3%ADk&amp;author=J%20Lukes&amp;author=PJ%20Keeling&amp;publication_year=2010&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A.&amp;volume=107&amp;pages=10949-10954" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1073/pnas.1003335107" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1003335107" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1003335107"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/20534454" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B21" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B21"><div class="ref false"><div role="msu189-B21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Janouskovec</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Sobotka</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Lai</div> <div class="given-names">DH</div></div>,&nbsp; <div class="name"><div class="surname">Flegontov</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Koník</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Komenda</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Ali</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Prásil</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Pain</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Oborník</div> <div class="given-names">M</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Split photosystem protein, linear-mapping topology, and growth of structural complexity in the plastid genome of <em>Chromera velia</em></div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">30</div>&nbsp;(pg.&nbsp;<div class="fpage">2447</div>-<div class="lpage">2462</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Split%20photosystem%20protein%2C%20linear-mapping%20topology%2C%20and%20growth%20of%20structural%20complexity%20in%20the%20plastid%20genome%20of%20Chromera%20velia&amp;author=J%20Janouskovec&amp;author=R%20Sobotka&amp;author=DH%20Lai&amp;author=P%20Flegontov&amp;author=P%20Kon%C3%ADk&amp;author=J%20Komenda&amp;author=S%20Ali&amp;author=O%20Pr%C3%A1sil&amp;author=A%20Pain&amp;author=M%20Oborn%C3%ADk&amp;publication_year=2013&amp;journal=Mol%20Biol%20Evol.&amp;volume=30&amp;pages=2447-2462" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/molbev/mst144" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst144" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmst144"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/23974208" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B22" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B22"><div class="ref false"><div role="msu189-B22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lange</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Sement</div> <div class="given-names">FM</div></div>,&nbsp; <div class="name"><div class="surname">Canaday</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Gagliardi</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">Polyadenylation-assisted RNA degradation processes in plants</div>,&nbsp;<div class="source">Trends Plant Sci.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">497</div>-<div class="lpage">504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Polyadenylation-assisted%20RNA%20degradation%20processes%20in%20plants&amp;author=H%20Lange&amp;author=FM%20Sement&amp;author=J%20Canaday&amp;author=D%20Gagliardi&amp;publication_year=2009&amp;journal=Trends%20Plant%20Sci.&amp;volume=14&amp;pages=497-504" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1016/j.tplants.2009.06.007" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2009.06.007" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tplants.2009.06.007"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/19716749" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B23" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B23"><div class="ref false"><div role="msu189-B23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larson</div> <div class="given-names">EM</div></div>,&nbsp; <div class="name"><div class="surname">Obrien</div> <div class="given-names">CM</div></div>,&nbsp; <div class="name"><div class="surname">Zhu</div> <div class="given-names">GH</div></div>,&nbsp; <div class="name"><div class="surname">Spreitzer</div> <div class="given-names">RJ</div></div>,&nbsp; <div class="name"><div class="surname">Portis</div> <div class="given-names">AR</div></div>.&nbsp;</div><div class="article-title">Specificity
 for activase is changed by a Pro-89 to Arg substitution in the large 
subunit of ribulose-1,5-bisphosphate carboxylase/oxygenase</div>,&nbsp;<div class="source">J Biol Chem.</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">272</div>&nbsp;(pg.&nbsp;<div class="fpage">17033</div>-<div class="lpage">17037</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Specificity%20for%20activase%20is%20changed%20by%20a%20Pro-89%20to%20Arg%20substitution%20in%20the%20large%20subunit%20of%20ribulose-1%2C5-bisphosphate%20carboxylase%2Foxygenase&amp;author=EM%20Larson&amp;author=CM%20Obrien&amp;author=GH%20Zhu&amp;author=RJ%20Spreitzer&amp;author=AR%20Portis&amp;publication_year=1997&amp;journal=J%20Biol%20Chem.&amp;volume=272&amp;pages=17033-17037" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1074/jbc.272.27.17033" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.272.27.17033" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1074%2Fjbc.272.27.17033"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/9202018" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B24" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B24"><div class="ref false"><div role="msu189-B24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">CH</div></div>,&nbsp; <div class="name"><div class="surname">Salvucci</div> <div class="given-names">ME</div></div>,&nbsp; <div class="name"><div class="surname">Portis</div> <div class="given-names">AR</div></div>.&nbsp;</div><div class="article-title">Two residues of rubisco activase involved in recognition of the rubisco substrate</div>,&nbsp;<div class="source">J Biol Chem.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">280</div>&nbsp;(pg.&nbsp;<div class="fpage">24864</div>-<div class="lpage">24869</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Two%20residues%20of%20rubisco%20activase%20involved%20in%20recognition%20of%20the%20rubisco%20substrate&amp;author=CH%20Li&amp;author=ME%20Salvucci&amp;author=AR%20Portis&amp;publication_year=2005&amp;journal=J%20Biol%20Chem.&amp;volume=280&amp;pages=24864-24869" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1074/jbc.M503547200" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M503547200" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1074%2Fjbc.M503547200"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/15866868" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B25" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B25"><div class="ref false"><div role="msu189-B25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">YG</div></div>,&nbsp; <div class="name"><div class="surname">Mitsukawa</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Oosumi</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Whittier</div> <div class="given-names">RF</div></div>.&nbsp;</div><div class="article-title">Efficient isolation and mapping of <em>Arabidopsis thaliana</em> T-DNA insert junctions by thermal asymmetric interlaced PCR</div>,&nbsp;<div class="source">Plant J.</div>&nbsp;,&nbsp;<div class="year">1995</div>,&nbsp;vol.&nbsp;<div class="volume">8</div>&nbsp;(pg.&nbsp;<div class="fpage">457</div>-<div class="lpage">463</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Efficient%20isolation%20and%20mapping%20of%20Arabidopsis%20thaliana%20T-DNA%20insert%20junctions%20by%20thermal%20asymmetric%20interlaced%20PCR&amp;author=YG%20Liu&amp;author=N%20Mitsukawa&amp;author=T%20Oosumi&amp;author=RF%20Whittier&amp;publication_year=1995&amp;journal=Plant%20J.&amp;volume=8&amp;pages=457-463" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1046/j.1365-313X.1995.08030457.x" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1995.08030457.x" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1365-313X.1995.08030457.x"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/7550382" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B26" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B26"><div class="ref false"><div role="msu189-B26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Chikuni</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Yabuki</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Takishita</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Kawachi</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Nakayama</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Inouye</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Hashimoto</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Inagaki</div> <div class="given-names">Y</div></div>.&nbsp;</div><div class="article-title">Green-colored plastids in the dinoflagellate genus <em>Lepidodinium</em> are of core chlorophyte origin</div>,&nbsp;<div class="source">Protist</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">162</div>&nbsp;(pg.&nbsp;<div class="fpage">268</div>-<div class="lpage">276</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Green-colored%20plastids%20in%20the%20dinoflagellate%20genus%20Lepidodinium%20are%20of%20core%20chlorophyte%20origin&amp;author=T%20Matsumoto&amp;author=F%20Shinozaki&amp;author=T%20Chikuni&amp;author=A%20Yabuki&amp;author=K%20Takishita&amp;author=M%20Kawachi&amp;author=T%20Nakayama&amp;author=I%20Inouye&amp;author=T%20Hashimoto&amp;author=Y%20Inagaki&amp;publication_year=2011&amp;journal=Protist&amp;volume=162&amp;pages=268-276" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1016/j.protis.2010.07.001" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.protis.2010.07.001" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.protis.2010.07.001"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/20829107" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B27" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B27"><div class="ref false"><div role="msu189-B27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minge</div> <div class="given-names">MA</div></div>,&nbsp; <div class="name"><div class="surname">Shalchian-Tabrizi</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Torresen</div> <div class="given-names">OK</div></div>,&nbsp; <div class="name"><div class="surname">Takishita</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Probert</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Inagaki</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Klaveness</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Jakobsen</div> <div class="given-names">KS</div></div>.&nbsp;</div><div class="article-title">A phylogenetic mosaic plastid proteome and unusual plastid-targeting signals in the green-colored dinoflagellate <em>Lepidodinium chlorophorum</em></div>,&nbsp;<div class="source">BMC Evol Biol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">10</div>&nbsp;pg.&nbsp;<div class="fpage">191</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=A%20phylogenetic%20mosaic%20plastid%20proteome%20and%20unusual%20plastid-targeting%20signals%20in%20the%20green-colored%20dinoflagellate%20Lepidodinium%20chlorophorum&amp;author=MA%20Minge&amp;author=K%20Shalchian-Tabrizi&amp;author=OK%20Torresen&amp;author=K%20Takishita&amp;author=I%20Probert&amp;author=Y%20Inagaki&amp;author=D%20Klaveness&amp;author=KS%20Jakobsen&amp;publication_year=2010&amp;journal=BMC%20Evol%20Biol.&amp;volume=10&amp;pages=191" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1186/1471-2148-10-191" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-191" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2148-10-191"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/20565933" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B41" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B41"><div class="ref false"><div role="msu189-B41" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mungpakdee</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Shinzato</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Takeuchi</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Kawashima</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Koyanagi</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Hisata</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Tanaka</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Goto</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Fujie</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Lin</div> <div class="given-names">S</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Massive gene transfer and extensive RNA editing of a symbiotic dinoflagellate plastid genome</div>,&nbsp;<div class="source">Genom Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2014</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">1408</div>-<div class="lpage">1422</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Massive%20gene%20transfer%20and%20extensive%20RNA%20editing%20of%20a%20symbiotic%20dinoflagellate%20plastid%20genome&amp;author=S%20Mungpakdee&amp;author=C%20Shinzato&amp;author=T%20Takeuchi&amp;author=T%20Kawashima&amp;author=R%20Koyanagi&amp;author=K%20Hisata&amp;author=M%20Tanaka&amp;author=H%20Goto&amp;author=M%20Fujie&amp;author=S%20Lin&amp;publication_year=2014&amp;journal=Genom%20Biol%20Evol.&amp;volume=6&amp;pages=1408-1422" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/gbe/evu109" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu109" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgbe%2Fevu109"> </span></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B28" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B28"><div class="ref false"><div role="msu189-B28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nash</div> <div class="given-names">EA</div></div>,&nbsp; <div class="name"><div class="surname">Barbrook</div> <div class="given-names">AC</div></div>,&nbsp; <div class="name"><div class="surname">Edwards-Stuart</div> <div class="given-names">RK</div></div>,&nbsp; <div class="name"><div class="surname">Bernhardt</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Howe</div> <div class="given-names">CJ</div></div>,&nbsp; <div class="name"><div class="surname">Nisbet</div> <div class="given-names">RER</div></div>.&nbsp;</div><div class="article-title">Organization of the mitochondrial genome in the dinoflagellate <em>Amphidinium carterae</em></div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">24</div>&nbsp;(pg.&nbsp;<div class="fpage">1528</div>-<div class="lpage">1536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Organization%20of%20the%20mitochondrial%20genome%20in%20the%20dinoflagellate%20Amphidinium%20carterae&amp;author=EA%20Nash&amp;author=AC%20Barbrook&amp;author=RK%20Edwards-Stuart&amp;author=K%20Bernhardt&amp;author=CJ%20Howe&amp;author=RER%20Nisbet&amp;publication_year=2007&amp;journal=Mol%20Biol%20Evol.&amp;volume=24&amp;pages=1528-1536" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/molbev/msm074" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm074" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsm074"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/17440175" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B29" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B29"><div class="ref false"><div role="msu189-B29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">Dang</div> <div class="given-names">YK</div></div>,&nbsp; <div class="name"><div class="surname">Filek</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Zhang</div> <div class="given-names">ZD</div></div>,&nbsp; <div class="name"><div class="surname">Yu</div> <div class="given-names">VWC</div></div>,&nbsp; <div class="name"><div class="surname">Ishida</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Green</div> <div class="given-names">BR</div></div>.&nbsp;</div><div class="article-title">Identification and transcription of transfer RNA genes in dinoflagellate plastid minicircles</div>,&nbsp;<div class="source">Gene</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">392</div>&nbsp;(pg.&nbsp;<div class="fpage">291</div>-<div class="lpage">298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Identification%20and%20transcription%20of%20transfer%20RNA%20genes%20in%20dinoflagellate%20plastid%20minicircles&amp;author=MJ%20Nelson&amp;author=YK%20Dang&amp;author=E%20Filek&amp;author=ZD%20Zhang&amp;author=VWC%20Yu&amp;author=K%20Ishida&amp;author=BR%20Green&amp;publication_year=2007&amp;journal=Gene&amp;volume=392&amp;pages=291-298" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1016/j.gene.2007.01.018" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.01.018" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2007.01.018"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/17350182" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B30" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B30"><div class="ref false"><div role="msu189-B30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Puerta</div> <div class="given-names">MVS</div></div>,&nbsp; <div class="name"><div class="surname">Bachvaroff</div> <div class="given-names">TR</div></div>,&nbsp; <div class="name"><div class="surname">Delwiche</div> <div class="given-names">CF</div></div>.&nbsp;</div><div class="article-title">The complete plastid genome sequence of the haptophyte <em>Emiliania huxleyi</em>: a comparison to other plastid genomes</div>,&nbsp;<div class="source">DNA Res.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">151</div>-<div class="lpage">156</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=The%20complete%20plastid%20genome%20sequence%20of%20the%20haptophyte%20Emiliania%20huxleyi%3A%20a%20comparison%20to%20other%20plastid%20genomes&amp;author=MVS%20Puerta&amp;author=TR%20Bachvaroff&amp;author=CF%20Delwiche&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=151-156" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/dnares/12.2.151" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.151" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2F12.2.151"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/16303746" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B31" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B31"><div class="ref false"><div role="msu189-B31" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shalchian-Tabrizi</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Skanseng</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Ronquist</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Klaveness</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Bachvaroff</div> <div class="given-names">TR</div></div>,&nbsp; <div class="name"><div class="surname">Delwiche</div> <div class="given-names">CF</div></div>,&nbsp; <div class="name"><div class="surname">Botnen</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Tengs</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Jakobsen</div> <div class="given-names">KS</div></div>.&nbsp;</div><div class="article-title">Heterotachy
 processes in rhodophyte-derived secondhand plastid genes: implications 
for addressing the origin and evolution of dinoflagellate plastids</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;(pg.&nbsp;<div class="fpage">1504</div>-<div class="lpage">1515</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Heterotachy%20processes%20in%20rhodophyte-derived%20secondhand%20plastid%20genes%3A%20implications%20for%20addressing%20the%20origin%20and%20evolution%20of%20dinoflagellate%20plastids&amp;author=K%20Shalchian-Tabrizi&amp;author=M%20Skanseng&amp;author=F%20Ronquist&amp;author=D%20Klaveness&amp;author=TR%20Bachvaroff&amp;author=CF%20Delwiche&amp;author=A%20Botnen&amp;author=T%20Tengs&amp;author=KS%20Jakobsen&amp;publication_year=2006&amp;journal=Mol%20Biol%20Evol.&amp;volume=23&amp;pages=1504-1515" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/molbev/msl011" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl011" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsl011"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/16699169" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B32" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B32"><div class="ref false"><div role="msu189-B32" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takano</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Hansen</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Fujita</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Horiguchi</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">Serial replacement of diatom endosymbionts in two freshwater dinoflagenates, <em>Peridiniopsis</em> spp. (Peridiniales, Dinophyceae)</div>,&nbsp;<div class="source">Phycologia</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">47</div>&nbsp;(pg.&nbsp;<div class="fpage">41</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Serial%20replacement%20of%20diatom%20endosymbionts%20in%20two%20freshwater%20dinoflagenates%2C%20Peridiniopsis%20spp.%20%28Peridiniales%2C%20Dinophyceae%29&amp;author=Y%20Takano&amp;author=G%20Hansen&amp;author=D%20Fujita&amp;author=T%20Horiguchi&amp;publication_year=2008&amp;journal=Phycologia&amp;volume=47&amp;pages=41-53" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.2216/07-36.1" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2216%2F07-36.1" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2216%2F07-36.1"> </span></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B33" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B33"><div class="ref false"><div role="msu189-B33" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takishita</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Kawachi</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Noel</div> <div class="given-names">M-H</div></div>,&nbsp; <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Kakizoe</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Watanabe</div> <div class="given-names">MM</div></div>,&nbsp; <div class="name"><div class="surname">Inouye</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Ishida</div> <div class="given-names">K-I</div></div>,&nbsp; <div class="name"><div class="surname">Hashimoto</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Inagaki</div> <div class="given-names">Y</div></div>.&nbsp;</div><div class="article-title">Origins of plastids and glyceraldehyde-3-phosphate dehydrogenase genes in the green-colored dinoflagellate <em>Lepidodinium chlorophorum</em></div>,&nbsp;<div class="source">Gene</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">410</div>&nbsp;(pg.&nbsp;<div class="fpage">26</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Origins%20of%20plastids%20and%20glyceraldehyde-3-phosphate%20dehydrogenase%20genes%20in%20the%20green-colored%20dinoflagellate%20Lepidodinium%20chlorophorum&amp;author=K%20Takishita&amp;author=M%20Kawachi&amp;author=M-H%20Noel&amp;author=T%20Matsumoto&amp;author=N%20Kakizoe&amp;author=MM%20Watanabe&amp;author=I%20Inouye&amp;author=K-I%20Ishida&amp;author=T%20Hashimoto&amp;author=Y%20Inagaki&amp;publication_year=2008&amp;journal=Gene&amp;volume=410&amp;pages=26-36" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1016/j.gene.2007.11.008" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.11.008" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2007.11.008"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/18191504" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B34" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B34"><div class="ref false"><div role="msu189-B34" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takishita</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Nakano</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Uchida</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Preliminary phylogenetic analysis of plastid-encoded genes from an anomalously pigmented dinoflagellate <em>Gymnodinium mikimotoi</em> (Gymnodiniales, Dinophyta)</div>,&nbsp;<div class="source">Phycol Res.</div>&nbsp;,&nbsp;<div class="year">1999</div>,&nbsp;vol.&nbsp;<div class="volume">47</div>&nbsp;(pg.&nbsp;<div class="fpage">257</div>-<div class="lpage">262</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Preliminary%20phylogenetic%20analysis%20of%20plastid-encoded%20genes%20from%20an%20anomalously%20pigmented%20dinoflagellate%20Gymnodinium%20mikimotoi%20%28Gymnodiniales%2C%20Dinophyta%29&amp;author=K%20Takishita&amp;author=K%20Nakano&amp;author=A%20Uchida&amp;publication_year=1999&amp;journal=Phycol%20Res.&amp;volume=47&amp;pages=257-262" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1111/pre.1999.47.issue-4" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpre.1999.47.issue-4" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fpre.1999.47.issue-4"> </span></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B35" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B35"><div class="ref false"><div role="msu189-B35" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Torarinsson</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Lindgreen</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">WAR: Webserver for aligning structural RNAs</div>,&nbsp;<div class="source">Nucl Acids Res.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">36</div>&nbsp;(pg.&nbsp;<div class="fpage">79</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=WAR%3A%20Webserver%20for%20aligning%20structural%20RNAs&amp;author=E%20Torarinsson&amp;author=S%20Lindgreen&amp;publication_year=2008&amp;journal=Nucl%20Acids%20Res.&amp;volume=36&amp;pages=79-84" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/nar/gkn275" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn275" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn275"> </span></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B36" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B36"><div class="ref false"><div role="msu189-B36" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">YL</div></div>,&nbsp; <div class="name"><div class="surname">Morse</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">Rampant polyuridylylation of plastid gene transcripts in the dinoflagellate <em>Lingulodinium</em></div>,&nbsp;<div class="source">Nucl Acids Res.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">34</div>&nbsp;(pg.&nbsp;<div class="fpage">613</div>-<div class="lpage">619</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Rampant%20polyuridylylation%20of%20plastid%20gene%20transcripts%20in%20the%20dinoflagellate%20Lingulodinium&amp;author=YL%20Wang&amp;author=D%20Morse&amp;publication_year=2006&amp;journal=Nucl%20Acids%20Res.&amp;volume=34&amp;pages=613-619" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/nar/gkj438" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj438" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkj438"> </span></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B37" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B37"><div class="ref false"><div role="msu189-B37" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zauner</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Greilinger</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Laatsch</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Kowallik</div> <div class="given-names">KV</div></div>,&nbsp; <div class="name"><div class="surname">Maier</div> <div class="given-names">UG</div></div>.&nbsp;</div><div class="article-title">Substitutional editing of transcripts from genes of cyanobacterial origin in the dinoflagellate <em>Ceratium horridum</em></div>,&nbsp;<div class="source">FEBS Lett.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">577</div>&nbsp;(pg.&nbsp;<div class="fpage">535</div>-<div class="lpage">538</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Substitutional%20editing%20of%20transcripts%20from%20genes%20of%20cyanobacterial%20origin%20in%20the%20dinoflagellate%20Ceratium%20horridum&amp;author=S%20Zauner&amp;author=D%20Greilinger&amp;author=T%20Laatsch&amp;author=KV%20Kowallik&amp;author=UG%20Maier&amp;publication_year=2004&amp;journal=FEBS%20Lett.&amp;volume=577&amp;pages=535-538" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1016/j.febslet.2004.10.060" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2004.10.060" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.febslet.2004.10.060"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/15556642" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B38" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B38"><div class="ref false"><div role="msu189-B38" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Green</div> <div class="given-names">BR</div></div>,&nbsp; <div class="name"><div class="surname">Cavalier-Smith</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">Single gene circles in dinoflagellate chloroplast genomes</div>,&nbsp;<div class="source">Nature</div>&nbsp;,&nbsp;<div class="year">1999</div>,&nbsp;vol.&nbsp;<div class="volume">400</div>&nbsp;(pg.&nbsp;<div class="fpage">155</div>-<div class="lpage">159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Single%20gene%20circles%20in%20dinoflagellate%20chloroplast%20genomes&amp;author=Z%20Zhang&amp;author=BR%20Green&amp;author=T%20Cavalier-Smith&amp;publication_year=1999&amp;journal=Nature&amp;volume=400&amp;pages=155-159" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1038/22099" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F22099" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F22099"> </span></div><div class="pub-id"><a href="http://www-ncbi-nlm-nih-gov.insb.bib.cnrs.fr/pubmed/10408440" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="msu189-B39" xmlns:helper="urn:XsltStringHelper" data-id="msu189-B39"><div class="ref false"><div role="msu189-B39" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zuker</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Mfold web server for nucleic acid folding and hybridization prediction</div>,&nbsp;<div class="source">Nucl Acids Res.</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">31</div>&nbsp;(pg.&nbsp;<div class="fpage">3406</div>-<div class="lpage">3415</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar_lookup?title=Mfold%20web%20server%20for%20nucleic%20acid%20folding%20and%20hybridization%20prediction&amp;author=M%20Zuker&amp;publication_year=2003&amp;journal=Nucl%20Acids%20Res.&amp;volume=31&amp;pages=3406-3415" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org.insb.bib.cnrs.fr/10.1093/nar/gkg595" target="">CrossRef</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg595" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkg595"> </span></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->




        

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution License 
(http://creativecommons.org/licenses/by/3.0/), which permits 
unrestricted reuse, distribution, and reproduction in any medium, 
provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        


    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2925780&amp;xsltPath=%7E/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download All Figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/9/10.1093_molbev_msu189/1/msu189_Supplementary_Data.zip?Expires=1513001585&amp;Signature=OHHgT43w1WXDYUG7p63bOJ7rN1s679ZHDkNWdLi%7E33p6LhBjso8IkZLAhaxMLoHm-E3k3lFMYRTeeK92l9mEmalFGyGQpclAggvzi39bHf19w51fv2II9YNNYfKp7v%7E4iusonhQ8yQuNUE2luLwqB9ZWKX7eoTh9lMWFcOsHYlyDO7w5npGVW-wlDjBYvBq7etY7MSK1l-GpCYpNTxAn2ITrr2R3W-LD9kUt3obH0rri8qCfMSbfBUgtLPczubatSWjBfL6QidVpNuw-jz9BFcN6%7EmMeiNpGeaegwiyy32eIHhkgLAAMC12uiXLdYRoc9FTpVLMZlarc7KhCqV4rxg__&amp;Key-Pair-Id=APKAIUCZBIA4LVPAVW3Q"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        <div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad2_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Supplement_Ad2_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Supplement_Ad2_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" frameborder="0" height="250" width="300"></iframe></div></div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">49</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com.insb.bib.cnrs.fr/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000343401100011&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-link-target="_blank" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/molbev/msu189" data-uuid="8dbf761c-4072-8bea-f67c-853e2543d0a9"><a target="" href="https://www.altmetric.com/details.php?domain=academic-oup-com.insb.bib.cnrs.fr&amp;citation_id=2556042" style="display:inline-block;">
        <img alt="Article has an altmetric score of 1" src="https://badges.altmetric.com/?size=128&amp;score=1&amp;types=ffffffff" style="border:0; margin:0; max-width: none;" height="64px" width="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2925780"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com.insb.bib.cnrs.fr/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000343401100011&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar-google-com.insb.bib.cnrs.fr/scholar?q=related:https%3a%2f%2facademic.oup.com%2fmbe%2farticle%2f31%2f9%2f2376%2f2925780">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
    <div class="article-cited-link-wrap web-of-science">
        <a href="http://gateway.webofknowledge.com.insb.bib.cnrs.fr/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000343401100011&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (8)</a>
    </div>    
    <div class="article-cited-link-wrap google-scholar-url">
        <a href="http://scholar.google.com.insb.bib.cnrs.fr/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F31%2F9%2F2376%2F2925780" target="">Google Scholar</a>
    </div>    
    <div class="article-cited-link-wrap crossref-citedby">
        <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/CrossRef-CitedBy/2925780">CrossRef</a>
    </div>

    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="#" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="#" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    

    <div class="widget-dynamic-entry-wrap">
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-article/doi/10.1093/molbev/msx320/4711175">Most m<sup>6</sup>A RNA modifications in protein-coding regions are evolutionarily unconserved and likely nonfunctional</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-article/doi/10.1093/molbev/msx317/4711174">The effects of sex-biased gene expression and X-linkage on rates of sequence evolution in <em>Drosophila</em></a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-article/doi/10.1093/molbev/msx315/4705838">A molecular portrait of de novo genes in yeasts</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-article/doi/10.1093/molbev/msx310/4705836">Diversity of translation initiation mechanisms across bacterial species is driven by environmental conditions and growth demands</a>
                </div> 
            </div>
            <div class="widget-dynamic-entry">
                <div class="fb-item-title">
                    <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/advance-article/doi/10.1093/molbev/msx308/4705835">Unbiased estimate of synonymous and non-synonymous substitution rates with non-stationary base composition</a>
                </div> 
            </div>
    </div>
    
    

</div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        <div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad3_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Supplement_Ad3_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Supplement_Ad3_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" frameborder="0" height="250" width="300"></iframe></div></div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="Genome-Wide Transcript Profiling Reveals the Coevolution of Plastid Gene Sequences and Transcript Processing Pathways in the Fucoxanthin Dinoflagellate Karlodinium veneficum" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        <div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad4_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Supplement_Ad4_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Supplement_Ad4_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" frameborder="0" height="90" width="728"></iframe></div></div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/About">About Molecular Biology and Evolution</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/Policies">Policies</a>
                            </li>
                            <li class="link-4">
                                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-5">
                                <a href="http://www.smbe.org/smbe/">Contact Us</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
                            <li class="link-2">
                                <a href="https://twitter.com/OfficialSMBE">Twitter</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe/subscribe">Purchase</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.oxfordjournals.org.insb.bib.cnrs.fr/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-5">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="http://medicine-and-health-careernetwork.oxfordjournals.org.insb.bib.cnrs.fr/">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic-oup-com.insb.bib.cnrs.fr/mbe" target="">
                <img id="footer-logo-MolecularBiologyandEvolution" class="journal-footer-affiliations-logo" src="mbe_f1-2062292105.svg" alt="Molecular Biology and Evolution">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1537-1719</li>
                                            <li>Print ISSN 0737-4038</li>
                    <li>Copyright © 2017 Society for Molecular Biology and Evolution</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com.insb.bib.cnrs.fr/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com.insb.bib.cnrs.fr/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com.insb.bib.cnrs.fr/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com.insb.bib.cnrs.fr/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com.insb.bib.cnrs.fr/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" height="42" width="150">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2017 Oxford University Press</li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/about_us/legal/privacy_policy">Privacy Policy</a></li>
    <li><a href="http://global.oup.com.insb.bib.cnrs.fr/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/about_us/legal/notices">Legal Notices</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic-oup-com.insb.bib.cnrs.fr/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic-oup-com.insb.bib.cnrs.fr/sign-in">Sign In</a> or <a href="https://academic-oup-com.insb.bib.cnrs.fr/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f9%2f2376%2f2925780">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="footerJS.js" type="text/javascript"></script>
<script src="OupCookiePolicyJS.js" type="text/javascript"></script>
<script src="MathJax.js" type="text/javascript"></script>
<script src="chartist.js" type="text/javascript"></script>
<script src="ArticlePageJS.js" type="text/javascript"></script>
<script src="addthis_widget.js" type="text/javascript"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh9" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.7a295a410262af12dfaaa96a.html#rand=0.07498004518434642&amp;iit=1512907987316&amp;tmr=load%3D1512907987277%26core%3D1512907987308%26main%3D1512907987311%26ifr%3D1512907987321&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic-oup-com.insb.bib.cnrs.fr&amp;dr=&amp;du=https%3A%2F%2Facademic-oup-com.insb.bib.cnrs.fr%2Fmbe%2Farticle%2F31%2F9%2F2376%2F2925780&amp;href=https%3A%2F%2Facademic-oup-com.insb.bib.cnrs.fr%2Fmbe%2Farticle%2F31%2F9%2F2376%2F2925780&amp;dt=Genome-Wide%20Transcript%20Profiling%20Reveals%20the%20Coevolution%20of%20Plastid%20Gene%20Sequences%20and%20Transcript%20Processing%20Pathways%20in%20the%20Fucoxanthin%20Dinoflagellate%20%20Karlodinium%20veneficum%20%20%7C%20Molecular%20Biology%20and%20Evolution%20%7C%20Oxford%20Academic&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5a2d24d3b7b5ff38&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DGenome-Wide%2520Transcript%2520Profiling%2520Reveals%2520the%2520Coevolution%2520of%2520Plastid%2520Gene%2520Sequences%2520and%2520Transcript%2520Processing%2520Pathways%2520in%2520the%2520Fucoxanthin%2520Dinoflagellate%2520%2520Karlodinium%2520veneficum%2520%2520%257C%2520Molecular%2520Biology%2520and%2520Evolution%2520%257C%2520Oxford%2520Academic%26description%3DPlastids%2520utilize%2520a%2520complex%2520gene%2520expression%2520machinery%252C%2520which%2520has%2520coevolved%2520with%2520the%2520underlying%2520genome%2520sequence.%2520Relatively%252C%2520little%2520is%2520known%2520about%2520the%2520genome-wide%2520evolution%2520of%2520transcript%2520processing%2520in%2520algal%2520plastids%2520that%2520have%2520undergone%2520complex%2520endosymbiotic%2520events.%2520We%2520present%2520the%2520first%2520genome-wide%2520study%2520of%2520transcript%2520processing%2520in%2520a%2520plastid%2520acquired%2520through%2520serial%2520endosymbiosis%252C%2520in%2520the%2520fucoxanthin-containing%2520dinoflagellate%2520Karlodinium%2520veneficum.%2520The%2520fucoxanthin%2520dinoflagellate%2520plastid%2520has%2520an%2520extremely%2520divergent%2520genome%2520and%2520utilizes%2520two%2520unusual%2520transcript%2520processing%2520pathways%252C%25203%25E2%2580%25B2-poly%28U%29%2520tail%2520addition%2520and%2520sequence%2520editing%252C%2520which%2520were%2520acquired%2520following%2520the%2520serial%2520endosymbiosis%2520event.%2520We%2520demonstrate%2520that%2520poly%28U%29%2520addition%2520and%2520sequence%2520editing%2520are%2520widespread%2520features%2520across%2520the%2520Karl.%2520veneficum%2520plastid%2520transcriptome%252C%2520whereas%2520other%2520dinoflagellate%2520plastid%2520lineages%2520that%2520have%2520arisen%2520through%2520independent%2520serial%2520endosymbiosis%2520events%2520do%2520not%2520utilize%2520either%2520RNA%2520processing%2520pathway.%2520These%2520pathways%2520constrain%2520the%2520effects%2520of%2520divergent%2520sequence%2520evolution%2520in%2520fucoxanthin%2520plastids%252C%2520for%2520example%2520by%2520correcting%2520mutations%2520in%2520the%2520genomic%2520sequence%2520that%2520would%2520otherwise%2520be%2520deleterious%252C%2520and%2520are%2520specifically%2520associated%2520with%2520transcripts%2520that%2520encode%2520functional%2520plastid%2520proteins%2520over%2520transcripts%2520of%2520recently%2520generated%2520pseudogenes.%2520These%2520pathways%2520may%2520have%2520additionally%2520facilitated%2520divergent%2520evolution%2520within%2520the%2520Karl.%2520veneficum%2520plastid.%2520Transcript%2520editing%252C%2520for%2520example%252C%2520has%2520contributed%2520to%2520the%2520evolution%2520of%2520a%2520novel%2520C-terminal%2520sequence%2520extension%2520on%2520the%2520Karl.%2520veneficum%2520AtpA%2520protein.%2520We%2520furthermore%2520provide%2520the%2520first%2520complete%2520sequence%2520of%2520an%2520episomal%2520minicircle%2520in%2520a%2520fucoxanthin%2520dinoflagellate%2520plastid%252C%2520which%2520contains%2520the%2520dnaK%2520gene%252C%2520and%2520gives%2520rise%2520to%2520polyuridylylated%2520and%2520edited%2520transcripts.%2520Our%2520results%2520indicate%2520that%2520RNA%2520processing%2520in%2520fucoxanthin%2520dinoflagellate%2520plastids%2520is%2520evolutionarily%2520dynamic%252C%2520coevolving%2520with%2520the%2520underlying%2520genome%2520sequence.%26url%3Dhttps%253A%252F%252Facademic-oup-com.insb.bib.cnrs.fr%252Fmbe%252Farticle%252F31%252F9%252F2376%252F2925780%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fmbe%252FIssue%252F34%252F12%252F2%252Fm_molbiolevol_34_12cover.png%253FExpires%253D1575979984%2526Signature%253DJR2w3ybdiL1sOtA3Wra03JPEEgKDMdKjnW7YP9Uo5clbc6jMiAjrVaX70rX9MqJQlH7IVuQhFUQvTF0EICX30qnlS%7E2xEpkq5BuRu4I4bPciFiF-hjwyBusxUK23WlBpxuAkaRoJ1EwC2D3Eojy7j7e%7EbqJVWsSOSRGyO9%7EsQjBy38u5mZcre-RsmhWHDCM9nD3ynK8y7AqSUWqwpRQeF0-oo8KqU58ADFgM4kUbIvLlPzxEm2KzvMkvEMKD8KdkZbIncWy%7EScXZhocQGABQeopg7zueRyfbdfmB0UM8T%7E4XdNKq-K4%7ERfGSZBXxuBYgVEJDtCrlE-sRpwIs7pbWDA__%2526Key-Pair-Id%253DAPKAIUCZBIA4LVPAVW3Q%26updated_time%3D2014-06-12%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.1.4-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<script type="text/javascript" src="msu189.json"></script></body></html>